Synthesis failed for java-compute

Hello! Autosynth couldn't regenerate java-compute. :broken_heart:

Here's the output from running `synth.py`:

```
synth.py', '--']
2020-09-01 14:20:37,029 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/30/sponge_log.log
2020-09-01 14:20:37,031 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-01 14:20:37,250 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-30
nothing to commit, working tree clean
2020-09-01 14:20:37,401 synthtool [DEBUG] > Ensuring dependencies.
2020-09-01 14:20:37,410 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-01 14:20:37,411 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-01 14:20:37,415 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-01 14:20:48,173 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.682s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-01 14:20:48,251 autosynth [ERROR] > Synthesis failed
2020-09-01 14:20:48,251 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-01 14:20:48,268 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-01 14:20:48,277 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e673ea8a-5db5-4e74-b9e9-780e97c69220/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
h.py', '--']
2020-09-02 01:20:27,819 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/32/sponge_log.log
2020-09-02 01:20:27,821 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-02 01:20:28,024 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-32
nothing to commit, working tree clean
2020-09-02 01:20:28,162 synthtool [DEBUG] > Ensuring dependencies.
2020-09-02 01:20:28,171 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-02 01:20:28,172 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-02 01:20:28,176 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-02 01:20:38,108 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.874s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-02 01:20:38,152 autosynth [ERROR] > Synthesis failed
2020-09-02 01:20:38,152 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-02 01:20:38,163 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-02 01:20:38,171 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e1025f31-8cda-4b9e-aa76-0983a8e7e5f3/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
a', 'synth.metadata', 'synth.py', '--']
2020-09-02 14:20:26,342 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/35/sponge_log.log
2020-09-02 14:20:26,344 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-02 14:20:26,561 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-35
nothing to commit, working tree clean
2020-09-02 14:20:26,712 synthtool [DEBUG] > Ensuring dependencies.
2020-09-02 14:20:26,722 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-02 14:20:26,723 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-02 14:20:26,727 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-02 14:20:36,587 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.792s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-02 14:20:36,647 autosynth [ERROR] > Synthesis failed
2020-09-02 14:20:36,648 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-02 14:20:36,659 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-02 14:20:36,669 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/09b5055b-c264-4360-b98e-04c4c842d5f9/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
synth.py', '--']
2020-09-03 01:25:38,256 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/38/sponge_log.log
2020-09-03 01:25:38,258 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-03 01:25:38,487 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-38
nothing to commit, working tree clean
2020-09-03 01:25:38,645 synthtool [DEBUG] > Ensuring dependencies.
2020-09-03 01:25:38,656 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-03 01:25:38,657 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-03 01:25:38,662 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-03 01:25:48,902 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.181s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-03 01:25:48,966 autosynth [ERROR] > Synthesis failed
2020-09-03 01:25:48,967 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-03 01:25:48,979 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-03 01:25:48,989 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/c33d21f1-c704-4e18-9358-4fc0e9f9f776/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-09-03 14:25:29,180 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/41/sponge_log.log
2020-09-03 14:25:29,183 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-03 14:25:29,427 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-41
nothing to commit, working tree clean
2020-09-03 14:25:29,586 synthtool [DEBUG] > Ensuring dependencies.
2020-09-03 14:25:29,597 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-03 14:25:29,598 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-03 14:25:29,603 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-03 14:25:39,969 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.296s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-03 14:25:40,058 autosynth [ERROR] > Synthesis failed
2020-09-03 14:25:40,059 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-03 14:25:40,076 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-03 14:25:40,092 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/25b42117-48ab-40c1-a6fb-a4a964867472/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
', 'synth.metadata', 'synth.py', '--']
2020-09-04 01:25:27,469 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/43/sponge_log.log
2020-09-04 01:25:27,472 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-04 01:25:27,696 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-43
nothing to commit, working tree clean
2020-09-04 01:25:27,848 synthtool [DEBUG] > Ensuring dependencies.
2020-09-04 01:25:27,858 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-04 01:25:27,859 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-04 01:25:27,863 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-04 01:25:38,093 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.169s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-04 01:25:38,171 autosynth [ERROR] > Synthesis failed
2020-09-04 01:25:38,172 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-04 01:25:38,187 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-04 01:25:38,201 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e7705faa-4f19-4596-8fbe-c9a34febd3c6/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-09-04 14:25:12,757 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/44/sponge_log.log
2020-09-04 14:25:12,759 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-04 14:25:12,986 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-44
nothing to commit, working tree clean
2020-09-04 14:25:13,141 synthtool [DEBUG] > Ensuring dependencies.
2020-09-04 14:25:13,151 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-04 14:25:13,152 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-04 14:25:13,156 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-04 14:25:24,289 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.059s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-04 14:25:24,380 autosynth [ERROR] > Synthesis failed
2020-09-04 14:25:24,380 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-04 14:25:24,398 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-04 14:25:24,413 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/827a9dc8-52ed-4bf7-9836-d12b3717473d/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
', 'synth.metadata', 'synth.py', '--']
2020-09-05 01:25:42,044 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/45/sponge_log.log
2020-09-05 01:25:42,046 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-05 01:25:42,280 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-45
nothing to commit, working tree clean
2020-09-05 01:25:42,436 synthtool [DEBUG] > Ensuring dependencies.
2020-09-05 01:25:42,447 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-05 01:25:42,448 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-05 01:25:42,452 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-05 01:25:52,823 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.305s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-05 01:25:52,921 autosynth [ERROR] > Synthesis failed
2020-09-05 01:25:52,922 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-05 01:25:52,939 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-05 01:25:52,954 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d16bc203-b7cb-4905-9969-adc3e504b699/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
h.py', '--']
2020-09-05 14:25:05,552 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/45/sponge_log.log
2020-09-05 14:25:05,554 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-05 14:25:05,775 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-45
nothing to commit, working tree clean
2020-09-05 14:25:05,928 synthtool [DEBUG] > Ensuring dependencies.
2020-09-05 14:25:05,939 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-05 14:25:05,940 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-05 14:25:05,944 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-05 14:25:15,773 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.774s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-05 14:25:15,833 autosynth [ERROR] > Synthesis failed
2020-09-05 14:25:15,834 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-05 14:25:15,846 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-05 14:25:15,857 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5fa52e8f-d388-4c38-b7af-71a0d55a4249/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-09-06 01:25:53,414 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/47/sponge_log.log
2020-09-06 01:25:53,417 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-06 01:25:53,674 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-47
nothing to commit, working tree clean
2020-09-06 01:25:53,842 synthtool [DEBUG] > Ensuring dependencies.
2020-09-06 01:25:53,853 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-06 01:25:53,854 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-06 01:25:53,858 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-06 01:26:04,359 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.430s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-06 01:26:04,436 autosynth [ERROR] > Synthesis failed
2020-09-06 01:26:04,437 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-06 01:26:04,449 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-06 01:26:04,460 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7e85e50c-35d5-440f-8ae0-2ac5cab87a02/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-09-06 14:25:18,770 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/48/sponge_log.log
2020-09-06 14:25:18,773 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-06 14:25:18,988 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-48
nothing to commit, working tree clean
2020-09-06 14:25:19,140 synthtool [DEBUG] > Ensuring dependencies.
2020-09-06 14:25:19,151 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-06 14:25:19,151 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-06 14:25:19,156 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-06 14:25:29,357 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.130s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-06 14:25:29,444 autosynth [ERROR] > Synthesis failed
2020-09-06 14:25:29,445 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-06 14:25:29,462 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-06 14:25:29,477 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4c406bd1-ee98-46f7-b1c8-ba4bc8fc1550/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-09-07 01:24:58,885 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/48/sponge_log.log
2020-09-07 01:24:58,887 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-07 01:24:59,109 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-48
nothing to commit, working tree clean
2020-09-07 01:24:59,260 synthtool [DEBUG] > Ensuring dependencies.
2020-09-07 01:24:59,270 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-07 01:24:59,271 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-07 01:24:59,275 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-07 01:25:09,876 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.546s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-07 01:25:09,934 autosynth [ERROR] > Synthesis failed
2020-09-07 01:25:09,935 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-07 01:25:09,946 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-07 01:25:09,956 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/fad6fb2d-cb84-453a-bc0b-e4dc3e6c55ed/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
', 'synth.metadata', 'synth.py', '--']
2020-09-07 14:24:57,828 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/50/sponge_log.log
2020-09-07 14:24:57,830 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-07 14:24:58,051 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-50
nothing to commit, working tree clean
2020-09-07 14:24:58,201 synthtool [DEBUG] > Ensuring dependencies.
2020-09-07 14:24:58,211 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-07 14:24:58,212 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-07 14:24:58,216 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-07 14:25:08,670 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.390s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-07 14:25:08,751 autosynth [ERROR] > Synthesis failed
2020-09-07 14:25:08,752 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-07 14:25:08,768 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-07 14:25:08,781 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/dbadc3fb-bf6b-4af2-ab77-7e3a846d96e8/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-09-08 01:25:42,586 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/51/sponge_log.log
2020-09-08 01:25:42,589 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-08 01:25:42,812 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-51
nothing to commit, working tree clean
2020-09-08 01:25:42,966 synthtool [DEBUG] > Ensuring dependencies.
2020-09-08 01:25:42,976 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-08 01:25:42,977 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-08 01:25:42,981 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-08 01:25:53,599 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.555s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-08 01:25:53,690 autosynth [ERROR] > Synthesis failed
2020-09-08 01:25:53,691 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0ca9fc0 test: fix more flaky network tests (#189)
2020-09-08 01:25:53,709 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-08 01:25:53,724 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/2c3d00d1-7009-47bc-b9f4-4aa6707299b4/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
EBUG] > log_file_path: /tmpfs/src/logs/java-compute/55/sponge_log.log
2020-09-08 14:42:13,788 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-08 14:42:14,006 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-55
nothing to commit, working tree clean
2020-09-08 14:42:14,153 synthtool [DEBUG] > Ensuring dependencies.
2020-09-08 14:42:14,163 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-08 14:42:14,164 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-08 14:42:14,167 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-08 14:42:23,970 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.736s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-08 14:42:24,043 autosynth [ERROR] > Synthesis failed
2020-09-08 14:42:24,043 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-08 14:42:24,059 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-08 14:42:24,073 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9f4d86d2-e590-420f-9223-70f186dccfee/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
EBUG] > log_file_path: /tmpfs/src/logs/java-compute/56/sponge_log.log
2020-09-09 01:29:02,224 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-09 01:29:02,442 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-56
nothing to commit, working tree clean
2020-09-09 01:29:02,591 synthtool [DEBUG] > Ensuring dependencies.
2020-09-09 01:29:02,602 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-09 01:29:02,603 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-09 01:29:02,607 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-09 01:29:12,644 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.984s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-09 01:29:12,701 autosynth [ERROR] > Synthesis failed
2020-09-09 01:29:12,701 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-09 01:29:12,713 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-09 01:29:12,723 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d3daad42-2df9-4cb8-a985-54c177a225aa/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 14:29:28,603 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/58/sponge_log.log
2020-09-09 14:29:28,606 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-09 14:29:28,846 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-58
nothing to commit, working tree clean
2020-09-09 14:29:29,010 synthtool [DEBUG] > Ensuring dependencies.
2020-09-09 14:29:29,023 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-09 14:29:29,024 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-09 14:29:29,029 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-09 14:29:39,410 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.319s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-09 14:29:39,505 autosynth [ERROR] > Synthesis failed
2020-09-09 14:29:39,505 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-09 14:29:39,523 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-09 14:29:39,538 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/39d3ee8f-8ff5-446b-9d42-fceff130305a/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 01:29:11,298 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/61/sponge_log.log
2020-09-10 01:29:11,300 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-10 01:29:11,524 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-61
nothing to commit, working tree clean
2020-09-10 01:29:11,681 synthtool [DEBUG] > Ensuring dependencies.
2020-09-10 01:29:11,692 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-10 01:29:11,693 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-10 01:29:11,697 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-10 01:29:21,856 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.101s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-10 01:29:21,935 autosynth [ERROR] > Synthesis failed
2020-09-10 01:29:21,935 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-10 01:29:21,951 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-10 01:29:21,966 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d9e96217-4f6e-4617-abfe-a9426ee22b75/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/64/sponge_log.log
2020-09-10 14:37:43,689 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-10 14:37:43,913 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-64
nothing to commit, working tree clean
2020-09-10 14:37:44,068 synthtool [DEBUG] > Ensuring dependencies.
2020-09-10 14:37:44,080 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-10 14:37:44,081 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-10 14:37:44,085 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-10 14:37:55,741 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.591s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-10 14:37:55,826 autosynth [ERROR] > Synthesis failed
2020-09-10 14:37:55,827 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-10 14:37:55,839 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-10 14:37:55,850 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7bd3109f-291e-465f-9df9-f3d8592f1b05/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
og
2020-09-11 01:35:19,949 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-11 01:35:20,166 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-66
nothing to commit, working tree clean
2020-09-11 01:35:20,314 synthtool [DEBUG] > Ensuring dependencies.
2020-09-11 01:35:20,325 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-11 01:35:20,326 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-11 01:35:20,330 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-11 01:35:30,758 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded)
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.367s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 16 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 16 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-11 01:35:30,816 autosynth [ERROR] > Synthesis failed
2020-09-11 01:35:30,817 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-11 01:35:30,829 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-11 01:35:30,839 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/6d12f39c-c7b3-40ec-b7aa-6dc341b90679/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 14:35:02,406 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/66/sponge_log.log
2020-09-11 14:35:02,408 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-11 14:35:02,615 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-66
nothing to commit, working tree clean
2020-09-11 14:35:02,755 synthtool [DEBUG] > Ensuring dependencies.
2020-09-11 14:35:02,765 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-11 14:35:02,766 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-11 14:35:02,770 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-11 14:35:13,667 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.828s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-11 14:35:13,742 autosynth [ERROR] > Synthesis failed
2020-09-11 14:35:13,742 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-11 14:35:13,759 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-11 14:35:13,769 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ba20ea17-806f-4f7a-aa1b-49f251bc9714/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/68/sponge_log.log
2020-09-12 01:35:30,317 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-12 01:35:30,540 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-68
nothing to commit, working tree clean
2020-09-12 01:35:30,693 synthtool [DEBUG] > Ensuring dependencies.
2020-09-12 01:35:30,705 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-12 01:35:30,706 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-12 01:35:30,710 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-12 01:35:41,957 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.187s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-12 01:35:42,019 autosynth [ERROR] > Synthesis failed
2020-09-12 01:35:42,019 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-12 01:35:42,031 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-12 01:35:42,042 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/264f7de6-6e24-433c-bb50-6531f218875e/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/68/sponge_log.log
2020-09-12 14:37:05,663 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-12 14:37:05,901 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-68
nothing to commit, working tree clean
2020-09-12 14:37:06,062 synthtool [DEBUG] > Ensuring dependencies.
2020-09-12 14:37:06,072 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-12 14:37:06,073 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-12 14:37:06,077 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-12 14:37:17,668 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.529s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-12 14:37:17,735 autosynth [ERROR] > Synthesis failed
2020-09-12 14:37:17,735 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-12 14:37:17,753 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-12 14:37:17,768 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7ce65735-133d-4c7e-8a77-6d503160029a/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/68/sponge_log.log
2020-09-13 01:36:06,088 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-13 01:36:06,313 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-68
nothing to commit, working tree clean
2020-09-13 01:36:06,468 synthtool [DEBUG] > Ensuring dependencies.
2020-09-13 01:36:06,479 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-13 01:36:06,480 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-13 01:36:06,485 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-13 01:36:17,968 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.418s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-13 01:36:18,043 autosynth [ERROR] > Synthesis failed
2020-09-13 01:36:18,043 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-13 01:36:18,061 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-13 01:36:18,076 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f559274a-d20e-483d-a774-ac38879c04d3/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 14:34:54,875 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/68/sponge_log.log
2020-09-13 14:34:54,877 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-13 14:34:55,098 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-68
nothing to commit, working tree clean
2020-09-13 14:34:55,247 synthtool [DEBUG] > Ensuring dependencies.
2020-09-13 14:34:55,258 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-13 14:34:55,259 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-13 14:34:55,264 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-13 14:35:06,505 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.182s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-13 14:35:06,583 autosynth [ERROR] > Synthesis failed
2020-09-13 14:35:06,583 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-13 14:35:06,600 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-13 14:35:06,615 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5fc7dd4d-91db-45e3-8340-b2dac6ef75ef/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 01:35:22,924 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/68/sponge_log.log
2020-09-14 01:35:22,926 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-14 01:35:23,143 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-68
nothing to commit, working tree clean
2020-09-14 01:35:23,294 synthtool [DEBUG] > Ensuring dependencies.
2020-09-14 01:35:23,305 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-14 01:35:23,306 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-14 01:35:23,310 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-14 01:35:34,437 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.068s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-14 01:35:34,512 autosynth [ERROR] > Synthesis failed
2020-09-14 01:35:34,512 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-14 01:35:34,528 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-14 01:35:34,542 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b38c5f85-3845-422c-8a6b-1447439e489d/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/70/sponge_log.log
2020-09-14 14:35:26,175 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-14 14:35:26,390 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-70
nothing to commit, working tree clean
2020-09-14 14:35:26,534 synthtool [DEBUG] > Ensuring dependencies.
2020-09-14 14:35:26,545 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-14 14:35:26,546 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-14 14:35:26,550 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-14 14:35:37,905 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.290s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-14 14:35:37,990 autosynth [ERROR] > Synthesis failed
2020-09-14 14:35:37,990 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-14 14:35:38,007 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-14 14:35:38,022 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b485cf90-4bd8-4bf6-80f8-696f605f3532/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/72/sponge_log.log
2020-09-15 01:42:46,876 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-15 01:42:47,098 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-72
nothing to commit, working tree clean
2020-09-15 01:42:47,250 synthtool [DEBUG] > Ensuring dependencies.
2020-09-15 01:42:47,260 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-15 01:42:47,261 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-15 01:42:47,266 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-15 01:42:58,542 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.214s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-15 01:42:58,613 autosynth [ERROR] > Synthesis failed
2020-09-15 01:42:58,614 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-15 01:42:58,630 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-15 01:42:58,645 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/19feb6d3-40a4-47b9-8f6c-2e7167da474a/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 14:34:35,270 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/74/sponge_log.log
2020-09-15 14:34:35,272 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-15 14:34:35,479 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-74
nothing to commit, working tree clean
2020-09-15 14:34:35,621 synthtool [DEBUG] > Ensuring dependencies.
2020-09-15 14:34:35,630 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-15 14:34:35,631 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-15 14:34:35,635 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-15 14:34:45,711 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.021s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-15 14:34:45,787 autosynth [ERROR] > Synthesis failed
2020-09-15 14:34:45,787 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-15 14:34:45,804 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-15 14:34:45,817 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5237c30e-85f0-4cf0-89ab-ef47ec1c8572/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/76/sponge_log.log
2020-09-16 01:40:17,747 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-16 01:40:17,982 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-76
nothing to commit, working tree clean
2020-09-16 01:40:18,139 synthtool [DEBUG] > Ensuring dependencies.
2020-09-16 01:40:18,150 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-16 01:40:18,151 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-16 01:40:18,155 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-16 01:40:29,098 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.867s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-16 01:40:29,198 autosynth [ERROR] > Synthesis failed
2020-09-16 01:40:29,199 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-16 01:40:29,219 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-16 01:40:29,235 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4448a658-3945-4c4a-ad8c-c27fbe9f8fdd/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/80/sponge_log.log
2020-09-16 14:37:59,528 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-16 14:37:59,823 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-80
nothing to commit, working tree clean
2020-09-16 14:37:59,989 synthtool [DEBUG] > Ensuring dependencies.
2020-09-16 14:38:00,001 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-16 14:38:00,002 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-16 14:38:00,008 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-16 14:38:12,385 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 12.296s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-16 14:38:12,488 autosynth [ERROR] > Synthesis failed
2020-09-16 14:38:12,488 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-16 14:38:12,507 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-16 14:38:12,519 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/2a25115a-a3a1-47ff-b9e8-1633953258a0/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/82/sponge_log.log
2020-09-17 01:35:50,787 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-17 01:35:51,020 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-82
nothing to commit, working tree clean
2020-09-17 01:35:51,175 synthtool [DEBUG] > Ensuring dependencies.
2020-09-17 01:35:51,186 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-17 01:35:51,187 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-17 01:35:51,192 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-17 01:36:03,328 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 12.075s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-17 01:36:03,387 autosynth [ERROR] > Synthesis failed
2020-09-17 01:36:03,387 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-17 01:36:03,400 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-17 01:36:03,411 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/24f3fe08-442c-4c71-bcc4-22bdd4d1d3a3/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/85/sponge_log.log
2020-09-17 14:51:55,958 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-17 14:51:56,173 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-85
nothing to commit, working tree clean
2020-09-17 14:51:56,318 synthtool [DEBUG] > Ensuring dependencies.
2020-09-17 14:51:56,330 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-17 14:51:56,331 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-17 14:51:56,335 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-17 14:52:08,095 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.693s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-17 14:52:08,180 autosynth [ERROR] > Synthesis failed
2020-09-17 14:52:08,180 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-17 14:52:08,197 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-17 14:52:08,207 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/a89ada08-4dfb-47d5-b8f7-042aafb7f10f/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/87/sponge_log.log
2020-09-18 02:05:47,676 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-18 02:05:47,888 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-87
nothing to commit, working tree clean
2020-09-18 02:05:48,038 synthtool [DEBUG] > Ensuring dependencies.
2020-09-18 02:05:48,048 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-18 02:05:48,049 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-18 02:05:48,053 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-18 02:05:59,689 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.569s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-18 02:05:59,782 autosynth [ERROR] > Synthesis failed
2020-09-18 02:05:59,782 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-18 02:05:59,799 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-18 02:05:59,812 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8ca90931-e804-4300-a5f8-bdb154647b97/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 14:38:47,949 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/89/sponge_log.log
2020-09-18 14:38:47,952 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-18 14:38:48,173 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-89
nothing to commit, working tree clean
2020-09-18 14:38:48,338 synthtool [DEBUG] > Ensuring dependencies.
2020-09-18 14:38:48,350 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-18 14:38:48,351 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-18 14:38:48,355 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-18 14:38:59,121 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.680s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-18 14:38:59,203 autosynth [ERROR] > Synthesis failed
2020-09-18 14:38:59,204 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-18 14:38:59,216 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-18 14:38:59,228 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8abedcdf-7fb2-4eb8-9396-088161c15ea7/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
9 01:38:26,330 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/90/sponge_log.log
2020-09-19 01:38:26,332 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-19 01:38:26,546 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-90
nothing to commit, working tree clean
2020-09-19 01:38:26,705 synthtool [DEBUG] > Ensuring dependencies.
2020-09-19 01:38:26,716 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-19 01:38:26,717 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-19 01:38:26,720 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-19 01:38:36,763 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.986s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-19 01:38:36,825 autosynth [ERROR] > Synthesis failed
2020-09-19 01:38:36,825 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-19 01:38:36,838 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-19 01:38:36,849 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4f317eba-f550-42ec-9ad5-d08faa1d398f/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/90/sponge_log.log
2020-09-19 14:37:20,768 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-19 14:37:20,982 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-90
nothing to commit, working tree clean
2020-09-19 14:37:21,127 synthtool [DEBUG] > Ensuring dependencies.
2020-09-19 14:37:21,137 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-19 14:37:21,138 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-19 14:37:21,142 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-19 14:37:32,060 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.857s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-19 14:37:32,117 autosynth [ERROR] > Synthesis failed
2020-09-19 14:37:32,117 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-19 14:37:32,135 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-19 14:37:32,146 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e1b85084-d83b-4179-aa86-5a92b2ca7b19/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 01:38:37,661 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/91/sponge_log.log
2020-09-20 01:38:37,665 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-20 01:38:37,889 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-91
nothing to commit, working tree clean
2020-09-20 01:38:38,045 synthtool [DEBUG] > Ensuring dependencies.
2020-09-20 01:38:38,057 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-20 01:38:38,058 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-20 01:38:38,062 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-20 01:38:49,325 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.202s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-20 01:38:49,419 autosynth [ERROR] > Synthesis failed
2020-09-20 01:38:49,419 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-20 01:38:49,432 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-20 01:38:49,443 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e27b03b6-1980-4f15-b148-49be97d88c32/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/91/sponge_log.log
2020-09-20 14:39:40,457 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-20 14:39:40,686 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-91
nothing to commit, working tree clean
2020-09-20 14:39:40,840 synthtool [DEBUG] > Ensuring dependencies.
2020-09-20 14:39:40,849 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-20 14:39:40,850 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-20 14:39:40,854 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-20 14:39:52,630 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.717s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-20 14:39:52,686 autosynth [ERROR] > Synthesis failed
2020-09-20 14:39:52,687 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-20 14:39:52,698 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-20 14:39:52,710 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f062382f-79f0-42ba-86b8-679b6e67616d/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/91/sponge_log.log
2020-09-21 01:38:52,818 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-21 01:38:53,051 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-91
nothing to commit, working tree clean
2020-09-21 01:38:53,224 synthtool [DEBUG] > Ensuring dependencies.
2020-09-21 01:38:53,235 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-21 01:38:53,236 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-21 01:38:53,241 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-21 01:39:04,606 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.293s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-21 01:39:04,700 autosynth [ERROR] > Synthesis failed
2020-09-21 01:39:04,700 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-21 01:39:04,718 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-21 01:39:04,732 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/1ad6c53e-6c38-4a70-97f6-5bef54c66bcf/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
h [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/96/sponge_log.log
2020-09-21 14:41:17,884 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-21 14:41:18,114 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-96
nothing to commit, working tree clean
2020-09-21 14:41:18,265 synthtool [DEBUG] > Ensuring dependencies.
2020-09-21 14:41:18,275 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-21 14:41:18,276 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-21 14:41:18,279 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-21 14:41:30,290 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.956s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-21 14:41:30,359 autosynth [ERROR] > Synthesis failed
2020-09-21 14:41:30,360 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-21 14:41:30,373 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-21 14:41:30,384 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/3f106841-5cc8-4ff7-88f5-971442197c3f/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
09-22 02:23:10,599 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-22 02:23:10,829 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-99
nothing to commit, working tree clean
2020-09-22 02:23:10,984 synthtool [DEBUG] > Ensuring dependencies.
2020-09-22 02:23:10,993 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-22 02:23:10,994 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-22 02:23:10,998 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-22 02:23:24,399 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Loading: 0 packages loaded
    currently loading: gapic/google/compute/v1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 13.341s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-22 02:23:24,459 autosynth [ERROR] > Synthesis failed
2020-09-22 02:23:24,459 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-22 02:23:24,471 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-22 02:23:24,481 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0b813512-0a4c-4833-b4a7-d4281972de25/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
G] > log_file_path: /tmpfs/src/logs/java-compute/102/sponge_log.log
2020-09-22 15:13:07,989 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-22 15:13:08,228 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-102
nothing to commit, working tree clean
2020-09-22 15:13:08,390 synthtool [DEBUG] > Ensuring dependencies.
2020-09-22 15:13:08,401 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-22 15:13:08,402 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-22 15:13:08,407 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-22 15:13:20,173 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.701s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-22 15:13:20,251 autosynth [ERROR] > Synthesis failed
2020-09-22 15:13:20,251 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8145520 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.9.0 (#197)
2020-09-22 15:13:20,264 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-22 15:13:20,276 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b12affff-e85c-44ff-9cad-bd06f777bd37/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
file_path: /tmpfs/src/logs/java-compute/105/sponge_log.log
2020-09-23 01:42:35,875 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-23 01:42:36,096 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-105
nothing to commit, working tree clean
2020-09-23 01:42:36,245 synthtool [DEBUG] > Ensuring dependencies.
2020-09-23 01:42:36,256 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-23 01:42:36,257 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-23 01:42:36,261 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-23 01:42:46,166 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.840s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-23 01:42:46,249 autosynth [ERROR] > Synthesis failed
2020-09-23 01:42:46,249 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a010ca4 deps: update dependency com.google.apis:google-api-services-compute to v1-rev20200811-1.30.10 (#193)
2020-09-23 01:42:46,267 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-23 01:42:46,283 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/16d494f8-3d5b-48d2-8519-a3f0981fed3a/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
-23 14:51:46,875 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/109/sponge_log.log
2020-09-23 14:51:46,877 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-23 14:51:47,110 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-109
nothing to commit, working tree clean
2020-09-23 14:51:47,271 synthtool [DEBUG] > Ensuring dependencies.
2020-09-23 14:51:47,282 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-23 14:51:47,283 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-23 14:51:47,287 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-23 14:51:57,260 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.915s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-23 14:51:57,346 autosynth [ERROR] > Synthesis failed
2020-09-23 14:51:57,346 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 43951a9 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.0 (#213)
2020-09-23 14:51:57,358 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-23 14:51:57,368 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d5869cdc-1d22-470d-9791-d8e49c2ab4c3/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/110/sponge_log.log
2020-09-24 01:27:46,516 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-24 01:27:46,721 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-110
nothing to commit, working tree clean
2020-09-24 01:27:46,862 synthtool [DEBUG] > Ensuring dependencies.
2020-09-24 01:27:46,872 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-24 01:27:46,873 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-24 01:27:46,877 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-24 01:27:57,055 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.119s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-24 01:27:57,135 autosynth [ERROR] > Synthesis failed
2020-09-24 01:27:57,135 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 43951a9 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.0 (#213)
2020-09-24 01:27:57,152 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-24 01:27:57,163 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8cc1af4d-bf40-4e43-aace-09969a81c16f/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
'--']
2020-09-24 15:05:28,937 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/113/sponge_log.log
2020-09-24 15:05:28,940 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-24 15:05:29,175 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-113
nothing to commit, working tree clean
2020-09-24 15:05:29,340 synthtool [DEBUG] > Ensuring dependencies.
2020-09-24 15:05:29,353 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-24 15:05:29,354 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-24 15:05:29,359 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-24 15:05:41,352 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.929s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-24 15:05:41,427 autosynth [ERROR] > Synthesis failed
2020-09-24 15:05:41,427 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-24 15:05:41,440 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-24 15:05:41,451 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8266a785-e3f0-42b4-a4ae-2b50884ff7cf/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
'--']
2020-09-25 01:43:45,235 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/118/sponge_log.log
2020-09-25 01:43:45,238 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-25 01:43:45,454 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-118
nothing to commit, working tree clean
2020-09-25 01:43:45,601 synthtool [DEBUG] > Ensuring dependencies.
2020-09-25 01:43:45,612 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-25 01:43:45,612 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-25 01:43:45,617 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-25 01:43:57,119 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.442s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-25 01:43:57,181 autosynth [ERROR] > Synthesis failed
2020-09-25 01:43:57,181 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-25 01:43:57,194 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-25 01:43:57,205 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/cd33b2f7-547e-47d3-b42f-1e9dad219ed0/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
'--']
2020-09-25 14:56:28,406 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/119/sponge_log.log
2020-09-25 14:56:28,409 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-25 14:56:28,631 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-119
nothing to commit, working tree clean
2020-09-25 14:56:28,784 synthtool [DEBUG] > Ensuring dependencies.
2020-09-25 14:56:28,795 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-25 14:56:28,796 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-25 14:56:28,801 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-25 14:56:40,284 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.421s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-25 14:56:40,358 autosynth [ERROR] > Synthesis failed
2020-09-25 14:56:40,358 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-25 14:56:40,370 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-25 14:56:40,380 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5b4ad1fe-5db7-46ef-b1c5-890a0edb7d62/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
nth.metadata', 'synth.py', '--']
2020-09-26 01:36:27,310 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/120/sponge_log.log
2020-09-26 01:36:27,312 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-26 01:36:27,539 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-120
nothing to commit, working tree clean
2020-09-26 01:36:27,709 synthtool [DEBUG] > Ensuring dependencies.
2020-09-26 01:36:27,720 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-26 01:36:27,721 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-26 01:36:27,725 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-26 01:36:38,083 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.303s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-26 01:36:38,166 autosynth [ERROR] > Synthesis failed
2020-09-26 01:36:38,166 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-26 01:36:38,178 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-26 01:36:38,190 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/bec9097e-6bcd-4276-a8b9-4680922df467/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
'--']
2020-09-26 14:36:15,224 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/120/sponge_log.log
2020-09-26 14:36:15,228 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-26 14:36:15,441 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-120
nothing to commit, working tree clean
2020-09-26 14:36:15,588 synthtool [DEBUG] > Ensuring dependencies.
2020-09-26 14:36:15,599 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-26 14:36:15,600 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-26 14:36:15,604 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-26 14:36:27,043 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.374s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-26 14:36:27,112 autosynth [ERROR] > Synthesis failed
2020-09-26 14:36:27,113 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-26 14:36:27,125 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-26 14:36:27,135 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d5b8ee44-e8ee-43f2-b6f2-c07ea6a54298/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/122/sponge_log.log
2020-09-27 01:36:10,010 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-27 01:36:10,234 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-122
nothing to commit, working tree clean
2020-09-27 01:36:10,397 synthtool [DEBUG] > Ensuring dependencies.
2020-09-27 01:36:10,409 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-27 01:36:10,410 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-27 01:36:10,414 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-27 01:36:20,265 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Loading: 0 packages loaded
    currently loading: gapic/google/compute/v1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.787s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-27 01:36:20,331 autosynth [ERROR] > Synthesis failed
2020-09-27 01:36:20,332 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-27 01:36:20,344 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-27 01:36:20,355 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/93eab562-151f-4964-b209-c6a2d9db608d/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
'--']
2020-09-27 14:36:02,382 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/122/sponge_log.log
2020-09-27 14:36:02,385 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-27 14:36:02,597 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-122
nothing to commit, working tree clean
2020-09-27 14:36:02,743 synthtool [DEBUG] > Ensuring dependencies.
2020-09-27 14:36:02,753 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-27 14:36:02,754 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-27 14:36:02,758 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-27 14:36:13,670 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.856s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-27 14:36:13,716 autosynth [ERROR] > Synthesis failed
2020-09-27 14:36:13,717 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-27 14:36:13,728 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-27 14:36:13,737 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/748b3ed1-fb9f-4386-9ec3-63b631d7f0c0/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-09-28 01:33:53,259 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/122/sponge_log.log
2020-09-28 01:33:53,261 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-28 01:33:53,462 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-122
nothing to commit, working tree clean
2020-09-28 01:33:53,612 synthtool [DEBUG] > Ensuring dependencies.
2020-09-28 01:33:53,621 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-28 01:33:53,622 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-28 01:33:53,626 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-28 01:34:03,282 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.610s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-28 01:34:03,333 autosynth [ERROR] > Synthesis failed
2020-09-28 01:34:03,333 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-28 01:34:03,344 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-28 01:34:03,353 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/3133ab8f-dee7-4d40-9416-8e4107897e3b/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
ynth.metadata', 'synth.py', '--']
2020-09-28 14:34:59,073 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/124/sponge_log.log
2020-09-28 14:34:59,075 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-28 14:34:59,293 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-124
nothing to commit, working tree clean
2020-09-28 14:34:59,441 synthtool [DEBUG] > Ensuring dependencies.
2020-09-28 14:34:59,450 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-28 14:34:59,451 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-28 14:34:59,455 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-28 14:35:09,494 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.988s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-28 14:35:09,552 autosynth [ERROR] > Synthesis failed
2020-09-28 14:35:09,553 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-28 14:35:09,568 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-28 14:35:09,577 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/1e3cd4ad-fd2d-4234-9911-e1f8b9d96de2/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
'--']
2020-09-29 01:36:03,008 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/126/sponge_log.log
2020-09-29 01:36:03,010 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-29 01:36:03,234 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-126
nothing to commit, working tree clean
2020-09-29 01:36:03,388 synthtool [DEBUG] > Ensuring dependencies.
2020-09-29 01:36:03,398 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-29 01:36:03,399 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-29 01:36:03,403 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-29 01:36:14,731 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.275s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-29 01:36:14,798 autosynth [ERROR] > Synthesis failed
2020-09-29 01:36:14,798 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-29 01:36:14,810 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-29 01:36:14,820 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4468ecfc-bd46-4e4e-bfeb-9a7f9dad792e/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
/tmpfs/src/logs/java-compute/128/sponge_log.log
2020-09-29 14:38:45,913 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-29 14:38:46,174 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-128
nothing to commit, working tree clean
2020-09-29 14:38:46,346 synthtool [DEBUG] > Ensuring dependencies.
2020-09-29 14:38:46,356 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-29 14:38:46,357 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-29 14:38:46,361 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-29 14:38:56,214 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded)
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.797s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-29 14:38:56,273 autosynth [ERROR] > Synthesis failed
2020-09-29 14:38:56,273 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-29 14:38:56,285 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-29 14:38:56,296 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d4efae04-0071-41b6-8a99-c1684be903f9/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
nth.metadata', 'synth.py', '--']
2020-09-30 02:06:10,005 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/130/sponge_log.log
2020-09-30 02:06:10,008 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-30 02:06:10,237 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-130
nothing to commit, working tree clean
2020-09-30 02:06:10,394 synthtool [DEBUG] > Ensuring dependencies.
2020-09-30 02:06:10,405 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-30 02:06:10,406 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-30 02:06:10,411 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-30 02:06:21,782 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.306s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-30 02:06:21,874 autosynth [ERROR] > Synthesis failed
2020-09-30 02:06:21,874 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-30 02:06:21,887 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-30 02:06:21,899 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0569c16d-a278-4261-99a5-cc4794f30ebd/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
nth.metadata', 'synth.py', '--']
2020-09-30 15:02:10,036 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/134/sponge_log.log
2020-09-30 15:02:10,038 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-30 15:02:10,258 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-134
nothing to commit, working tree clean
2020-09-30 15:02:10,405 synthtool [DEBUG] > Ensuring dependencies.
2020-09-30 15:02:10,415 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-09-30 15:02:10,416 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-09-30 15:02:10,420 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-09-30 15:02:21,905 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.425s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-09-30 15:02:21,969 autosynth [ERROR] > Synthesis failed
2020-09-30 15:02:21,969 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-09-30 15:02:21,982 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-30 15:02:21,994 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ba5c03b5-9f6c-4870-91e7-22fd13b7363f/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
23 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/135/sponge_log.log
2020-10-01 01:28:58,126 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-01 01:28:58,363 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-135
nothing to commit, working tree clean
2020-10-01 01:28:58,527 synthtool [DEBUG] > Ensuring dependencies.
2020-10-01 01:28:58,540 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-01 01:28:58,541 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-01 01:28:58,546 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-01 01:29:09,806 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.183s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-01 01:29:09,895 autosynth [ERROR] > Synthesis failed
2020-10-01 01:29:09,896 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 524ebad chore: add repo settings configuration (#215)
2020-10-01 01:29:09,911 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-01 01:29:09,924 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/c5240138-586a-4736-b233-c7ee0cb9c6e2/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
file_path: /tmpfs/src/logs/java-compute/137/sponge_log.log
2020-10-01 14:27:27,956 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-01 14:27:28,185 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-137
nothing to commit, working tree clean
2020-10-01 14:27:28,337 synthtool [DEBUG] > Ensuring dependencies.
2020-10-01 14:27:28,347 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-01 14:27:28,348 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-01 14:27:28,352 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-01 14:27:38,541 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded)
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.128s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-01 14:27:38,627 autosynth [ERROR] > Synthesis failed
2020-10-01 14:27:38,627 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-01 14:27:38,644 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-01 14:27:38,657 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f688eb48-aa56-4ecf-a44a-923c56b222fb/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
etadata', 'synth.metadata', 'synth.py', '--']
2020-10-02 01:27:57,485 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/140/sponge_log.log
2020-10-02 01:27:57,488 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-02 01:27:57,706 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-140
nothing to commit, working tree clean
2020-10-02 01:27:57,856 synthtool [DEBUG] > Ensuring dependencies.
2020-10-02 01:27:57,867 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-02 01:27:57,868 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-02 01:27:57,872 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-02 01:28:07,867 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.935s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-02 01:28:07,951 autosynth [ERROR] > Synthesis failed
2020-10-02 01:28:07,951 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-02 01:28:07,969 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-02 01:28:07,983 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/580c12ca-d12d-4db5-b0f0-6f2dca1b61de/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 'synth.py', '--']
2020-10-02 14:27:51,149 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/143/sponge_log.log
2020-10-02 14:27:51,152 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-02 14:27:51,372 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-143
nothing to commit, working tree clean
2020-10-02 14:27:51,523 synthtool [DEBUG] > Ensuring dependencies.
2020-10-02 14:27:51,534 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-02 14:27:51,535 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-02 14:27:51,539 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-02 14:28:01,582 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.982s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-02 14:28:01,642 autosynth [ERROR] > Synthesis failed
2020-10-02 14:28:01,643 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-02 14:28:01,655 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-02 14:28:01,666 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/6ffa0c70-430a-43ad-92a3-a1b34e1b0246/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
'synth.py', '--']
2020-10-03 01:28:18,686 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/145/sponge_log.log
2020-10-03 01:28:18,689 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-03 01:28:18,933 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-145
nothing to commit, working tree clean
2020-10-03 01:28:19,100 synthtool [DEBUG] > Ensuring dependencies.
2020-10-03 01:28:19,112 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-03 01:28:19,113 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-03 01:28:19,118 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-03 01:28:29,628 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.444s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-03 01:28:29,706 autosynth [ERROR] > Synthesis failed
2020-10-03 01:28:29,706 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-03 01:28:29,723 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-03 01:28:29,736 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/aa7f274a-0231-4c1b-986d-4d7befdfafa4/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
etadata', 'synth.metadata', 'synth.py', '--']
2020-10-03 14:27:22,868 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/147/sponge_log.log
2020-10-03 14:27:22,870 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-03 14:27:23,093 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-147
nothing to commit, working tree clean
2020-10-03 14:27:23,236 synthtool [DEBUG] > Ensuring dependencies.
2020-10-03 14:27:23,247 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-03 14:27:23,248 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-03 14:27:23,252 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-03 14:27:33,242 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.932s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-03 14:27:33,308 autosynth [ERROR] > Synthesis failed
2020-10-03 14:27:33,308 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-03 14:27:33,320 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-03 14:27:33,331 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e94b4410-f740-4c22-949c-048a427b7ebf/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
'synth.py', '--']
2020-10-04 01:27:27,073 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/148/sponge_log.log
2020-10-04 01:27:27,076 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-04 01:27:27,298 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-148
nothing to commit, working tree clean
2020-10-04 01:27:27,450 synthtool [DEBUG] > Ensuring dependencies.
2020-10-04 01:27:27,461 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-04 01:27:27,462 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-04 01:27:27,467 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-04 01:27:37,548 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.031s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-04 01:27:37,607 autosynth [ERROR] > Synthesis failed
2020-10-04 01:27:37,607 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-04 01:27:37,619 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-04 01:27:37,631 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/99d2a524-bf40-40ad-bb4f-eaa48eefc5fa/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
ta', 'synth.py', '--']
2020-10-04 14:28:30,439 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/149/sponge_log.log
2020-10-04 14:28:30,441 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-04 14:28:30,679 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-149
nothing to commit, working tree clean
2020-10-04 14:28:30,839 synthtool [DEBUG] > Ensuring dependencies.
2020-10-04 14:28:30,849 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-04 14:28:30,850 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-04 14:28:30,854 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-04 14:28:41,702 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.783s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-04 14:28:41,771 autosynth [ERROR] > Synthesis failed
2020-10-04 14:28:41,772 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-04 14:28:41,783 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-04 14:28:41,792 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7396152e-0a46-43ad-a9e7-649a22c9965e/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
tadata', 'synth.metadata', 'synth.py', '--']
2020-10-05 01:27:14,591 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/149/sponge_log.log
2020-10-05 01:27:14,592 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-05 01:27:14,807 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-149
nothing to commit, working tree clean
2020-10-05 01:27:14,954 synthtool [DEBUG] > Ensuring dependencies.
2020-10-05 01:27:14,963 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-05 01:27:14,964 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-05 01:27:14,967 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-05 01:27:25,113 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.088s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-05 01:27:25,176 autosynth [ERROR] > Synthesis failed
2020-10-05 01:27:25,176 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-05 01:27:25,187 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-05 01:27:25,196 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/74a423d6-efb2-4e23-84e3-8f9f5af33128/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
'synth.py', '--']
2020-10-05 14:27:57,626 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/150/sponge_log.log
2020-10-05 14:27:57,629 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-05 14:27:57,864 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-150
nothing to commit, working tree clean
2020-10-05 14:27:58,026 synthtool [DEBUG] > Ensuring dependencies.
2020-10-05 14:27:58,038 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-05 14:27:58,039 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-05 14:27:58,044 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-05 14:28:08,244 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.132s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-05 14:28:08,315 autosynth [ERROR] > Synthesis failed
2020-10-05 14:28:08,315 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-05 14:28:08,328 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-05 14:28:08,340 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5c1b6aa8-d444-42b5-b055-3f0f4257dcf5/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 'synth.py', '--']
2020-10-06 01:27:48,559 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/150/sponge_log.log
2020-10-06 01:27:48,561 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-06 01:27:48,781 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-150
nothing to commit, working tree clean
2020-10-06 01:27:48,930 synthtool [DEBUG] > Ensuring dependencies.
2020-10-06 01:27:48,941 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-06 01:27:48,942 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-06 01:27:48,947 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-06 01:27:59,021 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.999s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-06 01:27:59,108 autosynth [ERROR] > Synthesis failed
2020-10-06 01:27:59,109 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-06 01:27:59,121 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-06 01:27:59,131 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8f9052c0-f21a-44fe-a5c1-c6668f346a87/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
ean
2020-10-06 15:04:50,519 synthtool [DEBUG] > Ensuring dependencies.
2020-10-06 15:04:50,528 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-06 15:04:50,529 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-06 15:04:50,533 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-06 15:05:01,741 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
INFO: Call stack for the definition of repository 'rules_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_protobuf/protobuf_deps.bzl:42:9
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:25:1
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.147s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-06 15:05:01,824 autosynth [ERROR] > Synthesis failed
2020-10-06 15:05:01,824 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 07a1626 test: fix flaky ITDiskTest (#220)
2020-10-06 15:05:01,836 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-06 15:05:01,852 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d299163c-cf4b-4b05-8300-6d9cf898f75a/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/154/sponge_log.log
2020-10-07 01:27:56,315 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-07 01:27:56,547 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-154
nothing to commit, working tree clean
2020-10-07 01:27:56,703 synthtool [DEBUG] > Ensuring dependencies.
2020-10-07 01:27:56,715 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-07 01:27:56,716 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-07 01:27:56,720 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-07 01:28:07,414 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.584s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-07 01:28:07,489 autosynth [ERROR] > Synthesis failed
2020-10-07 01:28:07,490 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 12734fb chore(deps): update dependency com.google.cloud:libraries-bom to v12 (#223)
2020-10-07 01:28:07,503 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-07 01:28:07,515 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/05d82ac6-7ef6-43bd-bbf3-abdaa73a0bc3/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
G] > log_file_path: /tmpfs/src/logs/java-compute/156/sponge_log.log
2020-10-07 14:27:53,737 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-07 14:27:53,957 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-156
nothing to commit, working tree clean
2020-10-07 14:27:54,108 synthtool [DEBUG] > Ensuring dependencies.
2020-10-07 14:27:54,122 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-07 14:27:54,123 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-07 14:27:54,128 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-07 14:28:04,089 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.908s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-07 14:28:04,176 autosynth [ERROR] > Synthesis failed
2020-10-07 14:28:04,176 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2ab4b11 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.1 (#224)
2020-10-07 14:28:04,193 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-07 14:28:04,205 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b5f46f52-5a5d-4538-bf5c-f52b68795831/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/158/sponge_log.log
2020-10-08 01:27:50,548 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-08 01:27:50,773 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-158
nothing to commit, working tree clean
2020-10-08 01:27:50,926 synthtool [DEBUG] > Ensuring dependencies.
2020-10-08 01:27:50,937 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-08 01:27:50,938 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-08 01:27:50,942 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-08 01:28:01,211 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.213s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-08 01:28:01,295 autosynth [ERROR] > Synthesis failed
2020-10-08 01:28:01,295 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2ab4b11 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.1 (#224)
2020-10-08 01:28:01,314 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-08 01:28:01,328 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/eb6865f7-929f-49df-8dea-65cb56a3f83d/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/161/sponge_log.log
2020-10-08 14:27:46,301 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-08 14:27:46,545 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-161
nothing to commit, working tree clean
2020-10-08 14:27:46,710 synthtool [DEBUG] > Ensuring dependencies.
2020-10-08 14:27:46,723 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-08 14:27:46,724 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-08 14:27:46,730 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-08 14:27:58,436 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.625s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-08 14:27:58,524 autosynth [ERROR] > Synthesis failed
2020-10-08 14:27:58,524 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-08 14:27:58,536 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-08 14:27:58,547 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/56a5e0aa-f187-4267-b27c-29061fe24037/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
1:27:32,028 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/162/sponge_log.log
2020-10-09 01:27:32,030 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-09 01:27:32,257 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-162
nothing to commit, working tree clean
2020-10-09 01:27:32,412 synthtool [DEBUG] > Ensuring dependencies.
2020-10-09 01:27:32,422 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-09 01:27:32,423 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-09 01:27:32,427 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-09 01:27:42,260 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.778s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-09 01:27:42,321 autosynth [ERROR] > Synthesis failed
2020-10-09 01:27:42,321 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-09 01:27:42,332 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-09 01:27:42,342 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/00a36501-8116-4c54-854b-24df4a4b2f46/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/165/sponge_log.log
2020-10-09 14:28:32,340 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-09 14:28:32,595 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-165
nothing to commit, working tree clean
2020-10-09 14:28:32,765 synthtool [DEBUG] > Ensuring dependencies.
2020-10-09 14:28:32,775 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-09 14:28:32,776 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-09 14:28:32,781 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-09 14:28:44,928 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 12.091s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-09 14:28:45,000 autosynth [ERROR] > Synthesis failed
2020-10-09 14:28:45,001 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-09 14:28:45,012 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-09 14:28:45,024 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0caf0c30-f150-4178-bf58-f333b4e7ed00/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
1:27:13,153 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/166/sponge_log.log
2020-10-10 01:27:13,155 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-10 01:27:13,375 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-166
nothing to commit, working tree clean
2020-10-10 01:27:13,523 synthtool [DEBUG] > Ensuring dependencies.
2020-10-10 01:27:13,534 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-10 01:27:13,535 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-10 01:27:13,539 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-10 01:27:23,495 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.902s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-10 01:27:23,553 autosynth [ERROR] > Synthesis failed
2020-10-10 01:27:23,553 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-10 01:27:23,565 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-10 01:27:23,575 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/96f818d0-b29f-483c-b6ff-7efa4e67ffb1/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/167/sponge_log.log
2020-10-10 14:28:09,535 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-10 14:28:09,762 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-167
nothing to commit, working tree clean
2020-10-10 14:28:09,922 synthtool [DEBUG] > Ensuring dependencies.
2020-10-10 14:28:09,932 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-10 14:28:09,933 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-10 14:28:09,937 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-10 14:28:20,922 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.925s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-10 14:28:20,995 autosynth [ERROR] > Synthesis failed
2020-10-10 14:28:20,996 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-10 14:28:21,008 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-10 14:28:21,018 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/53d25a33-ab83-47f2-b008-d45fe5eacd7c/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/167/sponge_log.log
2020-10-11 01:27:42,087 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-11 01:27:42,324 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-167
nothing to commit, working tree clean
2020-10-11 01:27:42,485 synthtool [DEBUG] > Ensuring dependencies.
2020-10-11 01:27:42,497 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-11 01:27:42,498 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-11 01:27:42,503 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-11 01:27:53,018 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.445s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-11 01:27:53,090 autosynth [ERROR] > Synthesis failed
2020-10-11 01:27:53,091 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-11 01:27:53,104 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-11 01:27:53,116 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7c1e70b8-d2d7-4022-8af2-fdbe6d701662/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
:28:17,816 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/168/sponge_log.log
2020-10-11 14:28:17,818 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-11 14:28:18,055 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-168
nothing to commit, working tree clean
2020-10-11 14:28:18,213 synthtool [DEBUG] > Ensuring dependencies.
2020-10-11 14:28:18,222 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-11 14:28:18,223 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-11 14:28:18,227 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-11 14:28:28,938 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.650s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-11 14:28:29,008 autosynth [ERROR] > Synthesis failed
2020-10-11 14:28:29,009 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-11 14:28:29,026 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-11 14:28:29,035 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e30a5947-a034-4ce0-b525-ed56582a9d4d/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/168/sponge_log.log
2020-10-12 01:27:38,256 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-12 01:27:38,475 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-168
nothing to commit, working tree clean
2020-10-12 01:27:38,626 synthtool [DEBUG] > Ensuring dependencies.
2020-10-12 01:27:38,637 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-12 01:27:38,638 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-12 01:27:38,642 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-12 01:27:49,022 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.317s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-12 01:27:49,107 autosynth [ERROR] > Synthesis failed
2020-10-12 01:27:49,107 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-12 01:27:49,125 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-12 01:27:49,141 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/29fff33d-e84a-4ac6-92e0-775bee4fdd46/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
e/172/sponge_log.log
2020-10-12 14:27:24,324 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-12 14:27:24,549 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-172
nothing to commit, working tree clean
2020-10-12 14:27:24,705 synthtool [DEBUG] > Ensuring dependencies.
2020-10-12 14:27:24,716 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-12 14:27:24,717 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-12 14:27:24,721 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-12 14:27:34,301 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Loading: 0 packages loaded
    currently loading: gapic/google/compute/v1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.515s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-12 14:27:34,361 autosynth [ERROR] > Synthesis failed
2020-10-12 14:27:34,362 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-12 14:27:34,374 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-12 14:27:34,385 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/1db3b35c-6fb0-4e31-ab1f-d2f6f327745c/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/175/sponge_log.log
2020-10-13 01:28:02,860 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-13 01:28:03,098 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-175
nothing to commit, working tree clean
2020-10-13 01:28:03,258 synthtool [DEBUG] > Ensuring dependencies.
2020-10-13 01:28:03,269 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-13 01:28:03,270 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-13 01:28:03,275 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-13 01:28:13,804 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.462s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-13 01:28:13,872 autosynth [ERROR] > Synthesis failed
2020-10-13 01:28:13,872 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-13 01:28:13,885 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-13 01:28:13,896 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/525b6d17-873e-4c3b-bf11-bb86d27e468b/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/180/sponge_log.log
2020-10-13 15:20:27,973 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-13 15:20:28,196 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-180
nothing to commit, working tree clean
2020-10-13 15:20:28,348 synthtool [DEBUG] > Ensuring dependencies.
2020-10-13 15:20:28,359 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-13 15:20:28,360 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-13 15:20:28,364 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-13 15:20:39,468 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.039s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-13 15:20:39,526 autosynth [ERROR] > Synthesis failed
2020-10-13 15:20:39,526 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-13 15:20:39,538 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-13 15:20:39,546 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/154f5933-d9e3-4b3a-b54f-8f096c412698/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
:43:51,342 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/182/sponge_log.log
2020-10-14 01:43:51,344 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-14 01:43:51,559 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-182
nothing to commit, working tree clean
2020-10-14 01:43:51,705 synthtool [DEBUG] > Ensuring dependencies.
2020-10-14 01:43:51,714 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-14 01:43:51,715 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-14 01:43:51,719 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-14 01:44:02,716 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.933s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-14 01:44:02,779 autosynth [ERROR] > Synthesis failed
2020-10-14 01:44:02,779 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-14 01:44:02,791 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-14 01:44:02,801 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/900bf8e0-a9d1-400e-aa37-4b4e862b78b4/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
-14 14:29:02,073 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-14 14:29:02,337 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-186
nothing to commit, working tree clean
2020-10-14 14:29:02,517 synthtool [DEBUG] > Ensuring dependencies.
2020-10-14 14:29:02,531 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-14 14:29:02,533 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-14 14:29:02,538 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-14 14:29:14,071 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Loading: 0 packages loaded
    currently loading: gapic/google/compute/v1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.451s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-14 14:29:14,158 autosynth [ERROR] > Synthesis failed
2020-10-14 14:29:14,158 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-14 14:29:14,172 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-14 14:29:14,183 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9af7faf3-6e97-48b8-b691-962197a71016/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/188/sponge_log.log
2020-10-15 01:27:55,611 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-15 01:27:55,846 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-188
nothing to commit, working tree clean
2020-10-15 01:27:56,007 synthtool [DEBUG] > Ensuring dependencies.
2020-10-15 01:27:56,018 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-15 01:27:56,019 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-15 01:27:56,023 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-15 01:28:07,217 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.135s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-15 01:28:07,287 autosynth [ERROR] > Synthesis failed
2020-10-15 01:28:07,287 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-15 01:28:07,299 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-15 01:28:07,313 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/a338c6b7-1b4f-49e2-a853-ed9a1fae52ca/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
] > log_file_path: /tmpfs/src/logs/java-compute/192/sponge_log.log
2020-10-15 14:56:42,040 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-15 14:56:42,255 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-192
nothing to commit, working tree clean
2020-10-15 14:56:42,408 synthtool [DEBUG] > Ensuring dependencies.
2020-10-15 14:56:42,418 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-15 14:56:42,419 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-15 14:56:42,423 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-15 14:56:53,736 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.256s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-15 14:56:53,794 autosynth [ERROR] > Synthesis failed
2020-10-15 14:56:53,795 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-15 14:56:53,811 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-15 14:56:53,821 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 706, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 686, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/57db2c19-5ac3-4c0b-9eab-5ba604d919be/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/196/sponge_log.log
2020-10-16 01:57:50,589 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-16 01:57:50,818 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-196
nothing to commit, working tree clean
2020-10-16 01:57:50,976 synthtool [DEBUG] > Ensuring dependencies.
2020-10-16 01:57:50,987 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-16 01:57:50,988 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-16 01:57:50,992 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-16 01:58:02,560 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.505s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-16 01:58:02,628 autosynth [ERROR] > Synthesis failed
2020-10-16 01:58:02,628 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-16 01:58:02,641 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-16 01:58:02,652 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/82f153aa-4774-4f93-9e3a-5873761679b4/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/204/sponge_log.log
2020-10-16 14:28:05,693 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-16 14:28:05,914 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-204
nothing to commit, working tree clean
2020-10-16 14:28:06,068 synthtool [DEBUG] > Ensuring dependencies.
2020-10-16 14:28:06,080 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-16 14:28:06,081 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-16 14:28:06,085 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-16 14:28:16,676 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.519s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-16 14:28:16,739 autosynth [ERROR] > Synthesis failed
2020-10-16 14:28:16,740 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-16 14:28:16,751 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-16 14:28:16,763 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/80046b1e-c888-4fc5-adc8-945cf7a954b1/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/205/sponge_log.log
2020-10-17 01:32:24,698 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-17 01:32:24,938 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-205
nothing to commit, working tree clean
2020-10-17 01:32:25,100 synthtool [DEBUG] > Ensuring dependencies.
2020-10-17 01:32:25,111 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-17 01:32:25,112 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-17 01:32:25,117 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-17 01:32:36,047 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.874s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-17 01:32:36,096 autosynth [ERROR] > Synthesis failed
2020-10-17 01:32:36,096 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-17 01:32:36,113 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-17 01:32:36,124 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/de082443-a35b-4950-a8f9-0f2ab3b1ad2c/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
130 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/208/sponge_log.log
2020-10-17 14:29:37,132 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-17 14:29:37,347 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-208
nothing to commit, working tree clean
2020-10-17 14:29:37,499 synthtool [DEBUG] > Ensuring dependencies.
2020-10-17 14:29:37,509 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-17 14:29:37,510 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-17 14:29:37,514 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-17 14:29:47,703 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.136s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-17 14:29:47,760 autosynth [ERROR] > Synthesis failed
2020-10-17 14:29:47,760 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-17 14:29:47,772 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-17 14:29:47,781 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/845ec9a9-78e1-48c8-80a9-c2dddc82bb05/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
,017 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/208/sponge_log.log
2020-10-18 01:29:23,019 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-18 01:29:23,244 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-208
nothing to commit, working tree clean
2020-10-18 01:29:23,402 synthtool [DEBUG] > Ensuring dependencies.
2020-10-18 01:29:23,413 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-18 01:29:23,414 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-18 01:29:23,419 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-18 01:29:33,311 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.832s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-18 01:29:33,378 autosynth [ERROR] > Synthesis failed
2020-10-18 01:29:33,379 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-18 01:29:33,391 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-18 01:29:33,400 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/730220df-e628-4cfe-a982-d17c00fa68bb/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/208/sponge_log.log
2020-10-18 14:31:28,356 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-18 14:31:28,587 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-208
nothing to commit, working tree clean
2020-10-18 14:31:28,755 synthtool [DEBUG] > Ensuring dependencies.
2020-10-18 14:31:28,768 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-18 14:31:28,769 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-18 14:31:28,774 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-18 14:31:39,263 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.413s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-18 14:31:39,341 autosynth [ERROR] > Synthesis failed
2020-10-18 14:31:39,341 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-18 14:31:39,354 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-18 14:31:39,365 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/a578a81b-f946-4516-bfb5-eea8d8408a46/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
120 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/209/sponge_log.log
2020-10-19 01:30:09,122 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-19 01:30:09,353 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-209
nothing to commit, working tree clean
2020-10-19 01:30:09,508 synthtool [DEBUG] > Ensuring dependencies.
2020-10-19 01:30:09,518 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-19 01:30:09,519 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-19 01:30:09,523 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-19 01:30:20,207 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.625s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-19 01:30:20,294 autosynth [ERROR] > Synthesis failed
2020-10-19 01:30:20,295 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-19 01:30:20,307 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-19 01:30:20,319 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/1bc1367e-2e98-4447-b7e9-83c137a84447/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
14/sponge_log.log
2020-10-19 14:31:01,719 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-19 14:31:01,941 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-214
nothing to commit, working tree clean
2020-10-19 14:31:02,098 synthtool [DEBUG] > Ensuring dependencies.
2020-10-19 14:31:02,108 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-19 14:31:02,110 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-19 14:31:02,114 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-19 14:31:13,153 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Loading: 0 packages loaded
    currently loading: gapic/google/compute/v1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.969s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-19 14:31:13,233 autosynth [ERROR] > Synthesis failed
2020-10-19 14:31:13,234 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-19 14:31:13,251 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-19 14:31:13,266 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4835e044-0c14-49d0-a594-74609de1d9c9/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/217/sponge_log.log
2020-10-20 01:59:10,195 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-20 01:59:10,414 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-217
nothing to commit, working tree clean
2020-10-20 01:59:10,567 synthtool [DEBUG] > Ensuring dependencies.
2020-10-20 01:59:10,578 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-20 01:59:10,579 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-20 01:59:10,583 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-20 01:59:21,180 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.537s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-20 01:59:21,249 autosynth [ERROR] > Synthesis failed
2020-10-20 01:59:21,250 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-20 01:59:21,262 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-20 01:59:21,272 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/88c3385a-2961-4b74-9a45-bf9f27dc1e8c/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/226/sponge_log.log
2020-10-20 14:43:49,706 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-20 14:43:49,936 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-226
nothing to commit, working tree clean
2020-10-20 14:43:50,093 synthtool [DEBUG] > Ensuring dependencies.
2020-10-20 14:43:50,104 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-20 14:43:50,105 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-20 14:43:50,110 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-20 14:44:01,673 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.507s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-20 14:44:01,753 autosynth [ERROR] > Synthesis failed
2020-10-20 14:44:01,753 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-20 14:44:01,765 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-20 14:44:01,775 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0339287c-8bd0-41f5-9b2b-cd24c317f62b/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
> log_file_path: /tmpfs/src/logs/java-compute/233/sponge_log.log
2020-10-21 01:57:24,695 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-21 01:57:24,913 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-233
nothing to commit, working tree clean
2020-10-21 01:57:25,063 synthtool [DEBUG] > Ensuring dependencies.
2020-10-21 01:57:25,074 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-21 01:57:25,075 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-21 01:57:25,079 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-21 01:57:36,490 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.348s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-21 01:57:36,560 autosynth [ERROR] > Synthesis failed
2020-10-21 01:57:36,561 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-21 01:57:36,573 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-21 01:57:36,585 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/236d61e8-3b7a-4d32-ab80-0d6d5e08fd89/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/238/sponge_log.log
2020-10-21 14:33:59,785 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-21 14:34:00,014 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-238
nothing to commit, working tree clean
2020-10-21 14:34:00,170 synthtool [DEBUG] > Ensuring dependencies.
2020-10-21 14:34:00,181 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-21 14:34:00,182 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-21 14:34:00,186 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-21 14:34:11,098 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.854s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-21 14:34:11,163 autosynth [ERROR] > Synthesis failed
2020-10-21 14:34:11,163 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-21 14:34:11,175 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-21 14:34:11,185 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7190c05a-9947-4da8-aaea-64adccb6e46f/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/240/sponge_log.log
2020-10-22 01:30:07,634 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-22 01:30:07,894 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-240
nothing to commit, working tree clean
2020-10-22 01:30:08,064 synthtool [DEBUG] > Ensuring dependencies.
2020-10-22 01:30:08,074 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-22 01:30:08,075 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-22 01:30:08,080 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-22 01:30:19,154 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.009s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-22 01:30:19,214 autosynth [ERROR] > Synthesis failed
2020-10-22 01:30:19,214 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-22 01:30:19,226 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-22 01:30:19,237 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ceee4130-392c-4e75-9954-cd34f9f5c37c/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/249/sponge_log.log
2020-10-22 14:59:33,633 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-22 14:59:33,863 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-249
nothing to commit, working tree clean
2020-10-22 14:59:34,014 synthtool [DEBUG] > Ensuring dependencies.
2020-10-22 14:59:34,024 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-22 14:59:34,024 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-22 14:59:34,028 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-22 14:59:45,334 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.244s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-22 14:59:45,391 autosynth [ERROR] > Synthesis failed
2020-10-22 14:59:45,392 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-22 14:59:45,402 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-22 14:59:45,410 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/eb4eb2fc-bb36-479d-9fc0-f9e61df9f0a3/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
_file_path: /tmpfs/src/logs/java-compute/253/sponge_log.log
2020-10-23 01:39:08,690 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-23 01:39:08,902 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-253
nothing to commit, working tree clean
2020-10-23 01:39:09,047 synthtool [DEBUG] > Ensuring dependencies.
2020-10-23 01:39:09,057 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-23 01:39:09,058 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-23 01:39:09,062 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-23 01:39:19,478 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.365s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-23 01:39:19,540 autosynth [ERROR] > Synthesis failed
2020-10-23 01:39:19,540 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a1ce801 deps: update dependency com.google.cloud:google-cloud-shared-dependencies to v0.10.2 (#226)
2020-10-23 01:39:19,551 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-23 01:39:19,561 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/af182d0d-051e-41cf-9167-89f00deb4fc4/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/258/sponge_log.log
2020-10-23 14:39:48,325 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-23 14:39:48,549 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-258
nothing to commit, working tree clean
2020-10-23 14:39:48,700 synthtool [DEBUG] > Ensuring dependencies.
2020-10-23 14:39:48,711 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-23 14:39:48,712 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-23 14:39:48,716 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-23 14:39:59,730 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.953s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-23 14:39:59,794 autosynth [ERROR] > Synthesis failed
2020-10-23 14:39:59,794 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2d034be chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#237)
2020-10-23 14:39:59,806 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-23 14:39:59,816 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/be9b4c6d-e9b4-4685-83da-778386a58264/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```

2020-10-24 01:38:55,704 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/259/sponge_log.log
2020-10-24 01:38:55,706 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-24 01:38:55,916 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-259
nothing to commit, working tree clean
2020-10-24 01:38:56,060 synthtool [DEBUG] > Ensuring dependencies.
2020-10-24 01:38:56,071 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-24 01:38:56,072 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-24 01:38:56,076 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-24 01:39:07,206 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.075s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-24 01:39:07,267 autosynth [ERROR] > Synthesis failed
2020-10-24 01:39:07,267 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2d034be chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#237)
2020-10-24 01:39:07,279 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-24 01:39:07,288 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b1427179-de35-4105-830b-823ae87fbfe8/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
10-24 14:41:00,740 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-24 14:41:00,741 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-24 14:41:00,746 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-24 14:41:13,042 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
INFO: Call stack for the definition of repository 'rules_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_protobuf/protobuf_deps.bzl:42:9
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:25:1
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 12.222s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-24 14:41:13,114 autosynth [ERROR] > Synthesis failed
2020-10-24 14:41:13,115 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2d034be chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#237)
2020-10-24 14:41:13,131 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-24 14:41:13,142 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4b2012dc-bc34-4548-b8c9-bd06c510519e/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/260/sponge_log.log
2020-10-25 01:43:14,645 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-25 01:43:14,894 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-260
nothing to commit, working tree clean
2020-10-25 01:43:15,065 synthtool [DEBUG] > Ensuring dependencies.
2020-10-25 01:43:15,078 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-25 01:43:15,079 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-25 01:43:15,085 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-25 01:43:27,172 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 12.020s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-25 01:43:27,251 autosynth [ERROR] > Synthesis failed
2020-10-25 01:43:27,251 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2d034be chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#237)
2020-10-25 01:43:27,269 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-25 01:43:27,281 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b0f31c33-ee58-4f89-b7a7-df6f0b906a9e/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
0-10-25 14:39:07,962 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/260/sponge_log.log
2020-10-25 14:39:07,965 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-25 14:39:08,193 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-260
nothing to commit, working tree clean
2020-10-25 14:39:08,351 synthtool [DEBUG] > Ensuring dependencies.
2020-10-25 14:39:08,363 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-25 14:39:08,364 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-25 14:39:08,368 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-25 14:39:18,381 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.953s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-25 14:39:18,444 autosynth [ERROR] > Synthesis failed
2020-10-25 14:39:18,444 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2d034be chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#237)
2020-10-25 14:39:18,456 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-25 14:39:18,466 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/2c11861c-62b3-4669-8110-18af03716b42/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/260/sponge_log.log
2020-10-26 01:39:42,294 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-26 01:39:42,522 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-260
nothing to commit, working tree clean
2020-10-26 01:39:42,679 synthtool [DEBUG] > Ensuring dependencies.
2020-10-26 01:39:42,691 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-26 01:39:42,692 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-26 01:39:42,696 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-26 01:39:54,137 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.383s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-26 01:39:54,206 autosynth [ERROR] > Synthesis failed
2020-10-26 01:39:54,206 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2d034be chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#237)
2020-10-26 01:39:54,218 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-26 01:39:54,230 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/619af7d9-e625-43bb-9ee2-0621c4c06560/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/270/sponge_log.log
2020-10-26 14:40:55,158 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-26 14:40:55,389 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-270
nothing to commit, working tree clean
2020-10-26 14:40:55,550 synthtool [DEBUG] > Ensuring dependencies.
2020-10-26 14:40:55,562 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-26 14:40:55,563 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-26 14:40:55,568 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-26 14:41:07,077 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.435s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-26 14:41:07,164 autosynth [ERROR] > Synthesis failed
2020-10-26 14:41:07,164 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2d034be chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#237)
2020-10-26 14:41:07,177 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-26 14:41:07,188 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/eaa5a45b-d741-42dc-a886-188af362aae2/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```

2020-10-27 01:57:58,021 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/273/sponge_log.log
2020-10-27 01:57:58,023 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-27 01:57:58,250 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-273
nothing to commit, working tree clean
2020-10-27 01:57:58,405 synthtool [DEBUG] > Ensuring dependencies.
2020-10-27 01:57:58,416 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-27 01:57:58,417 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-27 01:57:58,421 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-27 01:58:09,564 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.079s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-27 01:58:09,632 autosynth [ERROR] > Synthesis failed
2020-10-27 01:58:09,633 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 2d034be chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#237)
2020-10-27 01:58:09,645 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-27 01:58:09,656 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5e2a7320-9071-4c7c-b457-b8fc67cdb8f8/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
945 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/278/sponge_log.log
2020-10-27 14:28:59,947 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-27 14:29:00,159 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-278
nothing to commit, working tree clean
2020-10-27 14:29:00,304 synthtool [DEBUG] > Ensuring dependencies.
2020-10-27 14:29:00,313 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-27 14:29:00,314 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-27 14:29:00,318 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-27 14:29:10,525 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.147s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-27 14:29:10,587 autosynth [ERROR] > Synthesis failed
2020-10-27 14:29:10,587 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at cd68820 build(deps): update dependency com.google.cloud:google-cloud-shared-config to v0.9.4 (#235)
2020-10-27 14:29:10,599 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-27 14:29:10,609 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/abdf26c4-a35b-41d3-8d29-93b067df7058/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
82/sponge_log.log
2020-10-28 01:30:40,639 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-28 01:30:40,875 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-282
nothing to commit, working tree clean
2020-10-28 01:30:41,037 synthtool [DEBUG] > Ensuring dependencies.
2020-10-28 01:30:41,048 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-28 01:30:41,050 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-28 01:30:41,054 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-28 01:30:52,414 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Loading: 0 packages loaded
    currently loading: gapic/google/compute/v1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.286s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-28 01:30:52,481 autosynth [ERROR] > Synthesis failed
2020-10-28 01:30:52,481 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 3689ef8 test(deps): update dependency junit:junit to v4.13.1 (#227)
2020-10-28 01:30:52,494 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-28 01:30:52,505 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/3eb7b1fe-b58c-4dc9-b748-428d6e3b56d0/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-10-28 14:42:19,644 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/287/sponge_log.log
2020-10-28 14:42:19,646 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-28 14:42:19,862 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-287
nothing to commit, working tree clean
2020-10-28 14:42:20,011 synthtool [DEBUG] > Ensuring dependencies.
2020-10-28 14:42:20,022 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-28 14:42:20,023 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-28 14:42:20,027 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-28 14:42:30,090 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.008s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-28 14:42:30,155 autosynth [ERROR] > Synthesis failed
2020-10-28 14:42:30,156 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 3689ef8 test(deps): update dependency junit:junit to v4.13.1 (#227)
2020-10-28 14:42:30,167 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-28 14:42:30,176 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/1a2298b4-1851-4b44-a226-c80426817cd6/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-10-29 01:31:30,986 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/290/sponge_log.log
2020-10-29 01:31:30,989 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-29 01:31:31,204 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-290
nothing to commit, working tree clean
2020-10-29 01:31:31,353 synthtool [DEBUG] > Ensuring dependencies.
2020-10-29 01:31:31,365 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-29 01:31:31,366 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-29 01:31:31,370 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-29 01:31:41,530 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.097s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-29 01:31:41,611 autosynth [ERROR] > Synthesis failed
2020-10-29 01:31:41,612 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 3689ef8 test(deps): update dependency junit:junit to v4.13.1 (#227)
2020-10-29 01:31:41,629 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-29 01:31:41,639 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/baee6ae8-059e-4a3b-a3e8-e49b8dcfb936/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
h.py', '--']
2020-10-29 14:31:13,954 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/294/sponge_log.log
2020-10-29 14:31:13,957 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-29 14:31:14,170 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-294
nothing to commit, working tree clean
2020-10-29 14:31:14,315 synthtool [DEBUG] > Ensuring dependencies.
2020-10-29 14:31:14,326 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-29 14:31:14,327 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-29 14:31:14,331 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-29 14:31:24,302 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.920s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-29 14:31:24,363 autosynth [ERROR] > Synthesis failed
2020-10-29 14:31:24,363 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 3689ef8 test(deps): update dependency junit:junit to v4.13.1 (#227)
2020-10-29 14:31:24,375 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-29 14:31:24,386 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f7ab3eb9-4799-49d8-9368-674dc223d7a2/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
.py', '--']
2020-10-30 01:31:19,794 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/297/sponge_log.log
2020-10-30 01:31:19,796 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-30 01:31:20,005 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-297
nothing to commit, working tree clean
2020-10-30 01:31:20,147 synthtool [DEBUG] > Ensuring dependencies.
2020-10-30 01:31:20,157 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-30 01:31:20,157 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-30 01:31:20,161 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-30 01:31:30,424 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.211s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-30 01:31:30,502 autosynth [ERROR] > Synthesis failed
2020-10-30 01:31:30,503 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 3689ef8 test(deps): update dependency junit:junit to v4.13.1 (#227)
2020-10-30 01:31:30,515 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-30 01:31:30,525 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8545974d-cffb-4074-a630-adbd51e3eb5e/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
 14:53:23,588 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/300/sponge_log.log
2020-10-30 14:53:23,590 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-30 14:53:23,805 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-300
nothing to commit, working tree clean
2020-10-30 14:53:23,954 synthtool [DEBUG] > Ensuring dependencies.
2020-10-30 14:53:23,965 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-30 14:53:23,966 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-30 14:53:23,970 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-30 14:53:35,281 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.256s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-30 14:53:35,337 autosynth [ERROR] > Synthesis failed
2020-10-30 14:53:35,337 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 3689ef8 test(deps): update dependency junit:junit to v4.13.1 (#227)
2020-10-30 14:53:35,349 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-30 14:53:35,358 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e07b6e89-ec48-4b2f-bfb3-3325e61d3771/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
[DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/302/sponge_log.log
2020-10-31 01:32:53,741 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-31 01:32:53,943 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-302
nothing to commit, working tree clean
2020-10-31 01:32:54,082 synthtool [DEBUG] > Ensuring dependencies.
2020-10-31 01:32:54,091 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-31 01:32:54,092 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-31 01:32:54,096 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-31 01:33:04,014 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.867s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-31 01:33:04,067 autosynth [ERROR] > Synthesis failed
2020-10-31 01:33:04,067 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-10-31 01:33:04,078 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-31 01:33:04,087 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9e390994-9568-4ab0-a8bd-b8b878c92d2b/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/302/sponge_log.log
2020-10-31 14:30:39,525 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-10-31 14:30:39,746 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-302
nothing to commit, working tree clean
2020-10-31 14:30:39,897 synthtool [DEBUG] > Ensuring dependencies.
2020-10-31 14:30:39,908 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-10-31 14:30:39,909 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-10-31 14:30:39,913 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-10-31 14:30:50,927 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.953s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 19 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-10-31 14:30:50,991 autosynth [ERROR] > Synthesis failed
2020-10-31 14:30:50,991 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-10-31 14:30:51,003 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-10-31 14:30:51,013 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/85a8e8a9-8a81-4fa7-a091-b1111aa656ae/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
01 01:30:47,229 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/302/sponge_log.log
2020-11-01 01:30:47,232 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-01 01:30:47,460 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-302
nothing to commit, working tree clean
2020-11-01 01:30:47,617 synthtool [DEBUG] > Ensuring dependencies.
2020-11-01 01:30:47,629 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-01 01:30:47,629 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-01 01:30:47,634 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-01 01:30:58,127 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.440s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-01 01:30:58,186 autosynth [ERROR] > Synthesis failed
2020-11-01 01:30:58,186 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-11-01 01:30:58,203 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-01 01:30:58,214 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e50e472d-95f0-4076-a0ee-92ab539f0ddf/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
[DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/302/sponge_log.log
2020-11-01 14:31:37,969 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-01 14:31:38,194 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-302
nothing to commit, working tree clean
2020-11-01 14:31:38,349 synthtool [DEBUG] > Ensuring dependencies.
2020-11-01 14:31:38,360 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-01 14:31:38,361 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-01 14:31:38,365 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-01 14:31:48,319 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.897s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-01 14:31:48,380 autosynth [ERROR] > Synthesis failed
2020-11-01 14:31:48,380 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-11-01 14:31:48,393 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-01 14:31:48,403 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f7647276-abfc-4f61-adb1-d12fcb8db957/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
nth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/303/sponge_log.log
2020-11-02 01:32:45,008 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-02 01:32:45,258 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-303
nothing to commit, working tree clean
2020-11-02 01:32:45,416 synthtool [DEBUG] > Ensuring dependencies.
2020-11-02 01:32:45,426 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-02 01:32:45,427 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-02 01:32:45,432 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-02 01:32:56,493 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.004s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-02 01:32:56,551 autosynth [ERROR] > Synthesis failed
2020-11-02 01:32:56,551 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-11-02 01:32:56,568 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-02 01:32:56,580 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/76d0f4f2-7164-4c96-8f73-ee8169b97e4f/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
02 14:36:15,410 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/311/sponge_log.log
2020-11-02 14:36:15,413 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-02 14:36:15,646 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-311
nothing to commit, working tree clean
2020-11-02 14:36:15,804 synthtool [DEBUG] > Ensuring dependencies.
2020-11-02 14:36:15,816 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-02 14:36:15,817 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-02 14:36:15,821 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-02 14:36:26,594 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.706s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-02 14:36:26,688 autosynth [ERROR] > Synthesis failed
2020-11-02 14:36:26,689 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-11-02 14:36:26,702 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-02 14:36:26,714 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/73f3164e-b6ed-410e-9f0d-ffb49f6a16f5/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
-02 16:40:02,551 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/313/sponge_log.log
2020-11-02 16:40:02,554 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-02 16:40:02,776 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-313
nothing to commit, working tree clean
2020-11-02 16:40:02,927 synthtool [DEBUG] > Ensuring dependencies.
2020-11-02 16:40:02,937 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-02 16:40:02,938 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-02 16:40:02,942 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-02 16:40:12,911 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.910s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-02 16:40:12,983 autosynth [ERROR] > Synthesis failed
2020-11-02 16:40:12,984 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-11-02 16:40:12,995 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-02 16:40:13,005 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/37d6057a-88b0-4c24-b2c4-e1e8a316f3d3/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/315/sponge_log.log
2020-11-03 01:29:38,808 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-03 01:29:39,022 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-315
nothing to commit, working tree clean
2020-11-03 01:29:39,182 synthtool [DEBUG] > Ensuring dependencies.
2020-11-03 01:29:39,192 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-03 01:29:39,193 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-03 01:29:39,197 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-03 01:29:49,402 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.137s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-03 01:29:49,464 autosynth [ERROR] > Synthesis failed
2020-11-03 01:29:49,464 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-11-03 01:29:49,476 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-03 01:29:49,486 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/6ce1e5fc-aed9-4052-8bbf-4477245a3095/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
-03 14:29:36,934 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/325/sponge_log.log
2020-11-03 14:29:36,936 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-03 14:29:37,159 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-325
nothing to commit, working tree clean
2020-11-03 14:29:37,323 synthtool [DEBUG] > Ensuring dependencies.
2020-11-03 14:29:37,334 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-03 14:29:37,335 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-03 14:29:37,339 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-03 14:29:47,044 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.652s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-03 14:29:47,110 autosynth [ERROR] > Synthesis failed
2020-11-03 14:29:47,110 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-11-03 14:29:47,123 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-03 14:29:47,133 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/45cb259e-1469-4922-94e9-7432c3d07045/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/328/sponge_log.log
2020-11-04 01:30:13,594 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-04 01:30:13,825 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-328
nothing to commit, working tree clean
2020-11-04 01:30:13,984 synthtool [DEBUG] > Ensuring dependencies.
2020-11-04 01:30:13,995 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-04 01:30:13,996 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-04 01:30:14,001 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-04 01:30:24,213 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.146s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-04 01:30:24,284 autosynth [ERROR] > Synthesis failed
2020-11-04 01:30:24,284 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 5fe4a9b chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#240)
2020-11-04 01:30:24,297 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-04 01:30:24,310 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d3b15a6a-0fd8-404b-ab4b-54f4e4afd0f2/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
ata', 'synth.metadata', 'synth.py', '--']
2020-11-04 14:49:16,877 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/336/sponge_log.log
2020-11-04 14:49:16,879 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-04 14:49:17,080 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-336
nothing to commit, working tree clean
2020-11-04 14:49:17,217 synthtool [DEBUG] > Ensuring dependencies.
2020-11-04 14:49:17,226 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-04 14:49:17,227 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-04 14:49:17,230 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-04 14:49:27,601 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.309s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-04 14:49:27,674 autosynth [ERROR] > Synthesis failed
2020-11-04 14:49:27,674 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 70df9c7 chore: release 0.119.0 (#181)
2020-11-04 14:49:27,686 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-04 14:49:27,694 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/390554c5-3ddc-46ad-a091-9f2993479248/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
data', 'synth.metadata', 'synth.py', '--']
2020-11-05 01:38:55,374 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/343/sponge_log.log
2020-11-05 01:38:55,376 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-05 01:38:55,581 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-343
nothing to commit, working tree clean
2020-11-05 01:38:55,724 synthtool [DEBUG] > Ensuring dependencies.
2020-11-05 01:38:55,733 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-05 01:38:55,734 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-05 01:38:55,738 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-05 01:39:05,786 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.987s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-05 01:39:05,844 autosynth [ERROR] > Synthesis failed
2020-11-05 01:39:05,845 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 70df9c7 chore: release 0.119.0 (#181)
2020-11-05 01:39:05,856 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-05 01:39:05,864 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f7f55ff2-b453-4080-ad10-06b9d1227efa/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
nth.py', '--']
2020-11-05 14:40:38,872 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/350/sponge_log.log
2020-11-05 14:40:38,876 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-05 14:40:39,140 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-350
nothing to commit, working tree clean
2020-11-05 14:40:39,326 synthtool [DEBUG] > Ensuring dependencies.
2020-11-05 14:40:39,342 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-05 14:40:39,343 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-05 14:40:39,349 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-05 14:40:51,873 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 12.446s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-05 14:40:51,962 autosynth [ERROR] > Synthesis failed
2020-11-05 14:40:51,962 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 70df9c7 chore: release 0.119.0 (#181)
2020-11-05 14:40:51,980 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-05 14:40:51,992 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4fe2485b-b7f2-44d9-80b3-ce91edb9d4a8/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
0-11-06 01:26:56,574 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/357/sponge_log.log
2020-11-06 01:26:56,577 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-06 01:26:56,808 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-357
nothing to commit, working tree clean
2020-11-06 01:26:56,969 synthtool [DEBUG] > Ensuring dependencies.
2020-11-06 01:26:56,980 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-06 01:26:56,981 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-06 01:26:56,986 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-06 01:27:06,809 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.763s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-06 01:27:06,865 autosynth [ERROR] > Synthesis failed
2020-11-06 01:27:06,866 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-06 01:27:06,877 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-06 01:27:06,885 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/51c20d2f-0156-410c-a746-e3bebcee0bcd/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
0-11-06 14:26:39,430 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/363/sponge_log.log
2020-11-06 14:26:39,432 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-06 14:26:39,646 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-363
nothing to commit, working tree clean
2020-11-06 14:26:39,791 synthtool [DEBUG] > Ensuring dependencies.
2020-11-06 14:26:39,801 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-06 14:26:39,802 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-06 14:26:39,806 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-06 14:26:49,414 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.550s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-06 14:26:49,480 autosynth [ERROR] > Synthesis failed
2020-11-06 14:26:49,481 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-06 14:26:49,492 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-06 14:26:49,503 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f2160b33-1145-453f-b78a-76038f86ffdd/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
tosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/367/sponge_log.log
2020-11-07 01:27:52,688 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-07 01:27:52,899 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-367
nothing to commit, working tree clean
2020-11-07 01:27:53,043 synthtool [DEBUG] > Ensuring dependencies.
2020-11-07 01:27:53,053 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-07 01:27:53,054 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-07 01:27:53,058 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-07 01:28:04,352 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.177s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-07 01:28:04,429 autosynth [ERROR] > Synthesis failed
2020-11-07 01:28:04,429 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-07 01:28:04,440 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-07 01:28:04,449 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/fcfdc677-8b7e-4479-aec0-e9b8e59f241e/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/368/sponge_log.log
2020-11-07 14:26:59,957 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-07 14:27:00,185 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-368
nothing to commit, working tree clean
2020-11-07 14:27:00,346 synthtool [DEBUG] > Ensuring dependencies.
2020-11-07 14:27:00,356 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-07 14:27:00,357 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-07 14:27:00,362 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-07 14:27:10,743 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.319s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 18 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-07 14:27:10,819 autosynth [ERROR] > Synthesis failed
2020-11-07 14:27:10,820 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-07 14:27:10,831 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-07 14:27:10,842 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/973ff63f-b75d-4868-a2e3-a130731ca71c/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/368/sponge_log.log
2020-11-08 01:26:40,263 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-08 01:26:40,484 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-368
nothing to commit, working tree clean
2020-11-08 01:26:40,636 synthtool [DEBUG] > Ensuring dependencies.
2020-11-08 01:26:40,646 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-08 01:26:40,647 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-08 01:26:40,650 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-08 01:26:50,983 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.270s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-08 01:26:51,044 autosynth [ERROR] > Synthesis failed
2020-11-08 01:26:51,044 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-08 01:26:51,056 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-08 01:26:51,066 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d1bd0774-a341-4dea-91eb-cb26ba7ee5aa/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/368/sponge_log.log
2020-11-08 14:26:56,441 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-08 14:26:56,671 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-368
nothing to commit, working tree clean
2020-11-08 14:26:56,829 synthtool [DEBUG] > Ensuring dependencies.
2020-11-08 14:26:56,840 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-08 14:26:56,841 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-08 14:26:56,846 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-08 14:27:06,971 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.060s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 20 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-08 14:27:07,038 autosynth [ERROR] > Synthesis failed
2020-11-08 14:27:07,038 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-08 14:27:07,051 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-08 14:27:07,062 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/235906ea-3aac-46b2-9d95-0844fa622a6a/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
0-11-09 01:27:02,089 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/368/sponge_log.log
2020-11-09 01:27:02,091 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-09 01:27:02,323 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-368
nothing to commit, working tree clean
2020-11-09 01:27:02,480 synthtool [DEBUG] > Ensuring dependencies.
2020-11-09 01:27:02,491 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-09 01:27:02,492 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-09 01:27:02,497 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-09 01:27:12,552 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.994s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-09 01:27:12,616 autosynth [ERROR] > Synthesis failed
2020-11-09 01:27:12,616 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-09 01:27:12,628 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-09 01:27:12,639 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/80ddbaa1-4140-42b6-892f-024fd5d543d1/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/376/sponge_log.log
2020-11-09 14:31:25,038 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-09 14:31:25,263 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-376
nothing to commit, working tree clean
2020-11-09 14:31:25,425 synthtool [DEBUG] > Ensuring dependencies.
2020-11-09 14:31:25,438 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-09 14:31:25,439 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-09 14:31:25,444 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-09 14:31:36,373 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.863s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-09 14:31:36,434 autosynth [ERROR] > Synthesis failed
2020-11-09 14:31:36,434 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-09 14:31:36,446 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-09 14:31:36,456 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/158686d3-7f16-451c-8967-b9a9aa6f2922/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/380/sponge_log.log
2020-11-10 01:53:17,314 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-10 01:53:17,529 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-380
nothing to commit, working tree clean
2020-11-10 01:53:17,685 synthtool [DEBUG] > Ensuring dependencies.
2020-11-10 01:53:17,697 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-10 01:53:17,698 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-10 01:53:17,702 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-10 01:53:28,978 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.218s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-10 01:53:29,067 autosynth [ERROR] > Synthesis failed
2020-11-10 01:53:29,067 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-10 01:53:29,079 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-10 01:53:29,092 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/56f6b42a-a80d-49a4-8def-3f7c5f01609c/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/384/sponge_log.log
2020-11-10 14:27:47,182 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-10 14:27:47,399 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-384
nothing to commit, working tree clean
2020-11-10 14:27:47,549 synthtool [DEBUG] > Ensuring dependencies.
2020-11-10 14:27:47,558 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-10 14:27:47,559 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-10 14:27:47,563 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-10 14:27:57,883 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.266s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-10 14:27:57,942 autosynth [ERROR] > Synthesis failed
2020-11-10 14:27:57,942 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-10 14:27:57,953 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-10 14:27:57,963 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/38c09443-a9bc-41cb-84a9-ec0e6d26cb95/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/388/sponge_log.log
2020-11-11 01:27:37,739 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-11 01:27:37,953 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-388
nothing to commit, working tree clean
2020-11-11 01:27:38,101 synthtool [DEBUG] > Ensuring dependencies.
2020-11-11 01:27:38,112 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-11 01:27:38,113 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-11 01:27:38,117 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-11 01:27:49,171 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.994s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-11 01:27:49,253 autosynth [ERROR] > Synthesis failed
2020-11-11 01:27:49,253 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-11 01:27:49,270 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-11 01:27:49,284 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4357c73c-318f-4224-b171-32004d0f6fe2/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
0-11-11 14:26:07,213 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/395/sponge_log.log
2020-11-11 14:26:07,215 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-11 14:26:07,420 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-395
nothing to commit, working tree clean
2020-11-11 14:26:07,561 synthtool [DEBUG] > Ensuring dependencies.
2020-11-11 14:26:07,571 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-11 14:26:07,572 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-11 14:26:07,575 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-11 14:26:17,097 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.461s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-11 14:26:17,153 autosynth [ERROR] > Synthesis failed
2020-11-11 14:26:17,154 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-11 14:26:17,165 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-11 14:26:17,175 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5752740d-1b1c-4a59-900d-e12163d7a919/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/398/sponge_log.log
2020-11-12 01:26:59,368 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-12 01:26:59,588 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-398
nothing to commit, working tree clean
2020-11-12 01:26:59,741 synthtool [DEBUG] > Ensuring dependencies.
2020-11-12 01:26:59,752 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-12 01:26:59,752 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-12 01:26:59,757 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-12 01:27:10,732 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.913s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-12 01:27:10,809 autosynth [ERROR] > Synthesis failed
2020-11-12 01:27:10,809 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-12 01:27:10,826 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-12 01:27:10,836 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d583e2ce-584e-4d3a-a4cd-4c6b6b7c3733/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/405/sponge_log.log
2020-11-12 14:50:46,927 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-12 14:50:47,156 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-405
nothing to commit, working tree clean
2020-11-12 14:50:47,310 synthtool [DEBUG] > Ensuring dependencies.
2020-11-12 14:50:47,321 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-12 14:50:47,322 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-12 14:50:47,326 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-12 14:50:59,023 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.627s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-12 14:50:59,112 autosynth [ERROR] > Synthesis failed
2020-11-12 14:50:59,113 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-12 14:50:59,125 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-12 14:50:59,135 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/c20efb24-7a7c-45d4-9989-31379a77571d/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
-11-13 01:36:40,939 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/407/sponge_log.log
2020-11-13 01:36:40,941 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-13 01:36:41,143 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-407
nothing to commit, working tree clean
2020-11-13 01:36:41,287 synthtool [DEBUG] > Ensuring dependencies.
2020-11-13 01:36:41,296 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-13 01:36:41,297 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-13 01:36:41,301 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-13 01:36:51,958 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.604s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-13 01:36:52,017 autosynth [ERROR] > Synthesis failed
2020-11-13 01:36:52,017 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-13 01:36:52,028 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-13 01:36:52,036 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/28eef474-a019-43ef-bb29-aaf65ec75a19/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
0-11-13 14:26:27,072 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/412/sponge_log.log
2020-11-13 14:26:27,074 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-13 14:26:27,291 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-412
nothing to commit, working tree clean
2020-11-13 14:26:27,438 synthtool [DEBUG] > Ensuring dependencies.
2020-11-13 14:26:27,448 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-13 14:26:27,449 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-13 14:26:27,453 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-13 14:26:36,668 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.156s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-13 14:26:36,723 autosynth [ERROR] > Synthesis failed
2020-11-13 14:26:36,724 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-13 14:26:36,735 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-13 14:26:36,744 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/30bf10d5-5381-4eee-ab95-75a1892097ce/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
0-11-14 01:34:03,770 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/417/sponge_log.log
2020-11-14 01:34:03,772 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-14 01:34:04,012 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-417
nothing to commit, working tree clean
2020-11-14 01:34:04,171 synthtool [DEBUG] > Ensuring dependencies.
2020-11-14 01:34:04,182 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-14 01:34:04,183 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-14 01:34:04,187 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-14 01:34:14,028 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.788s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-14 01:34:14,092 autosynth [ERROR] > Synthesis failed
2020-11-14 01:34:14,093 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-14 01:34:14,104 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-14 01:34:14,114 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/94022e06-72aa-4bec-bc8a-4740c405435e/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
-11-14 14:27:54,217 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/420/sponge_log.log
2020-11-14 14:27:54,219 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-14 14:27:54,439 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-420
nothing to commit, working tree clean
2020-11-14 14:27:54,590 synthtool [DEBUG] > Ensuring dependencies.
2020-11-14 14:27:54,600 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-14 14:27:54,601 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-14 14:27:54,605 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-14 14:28:04,850 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.192s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-14 14:28:04,923 autosynth [ERROR] > Synthesis failed
2020-11-14 14:28:04,923 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-14 14:28:04,936 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-14 14:28:04,947 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/15020e1b-e39c-4b0f-8cea-b5cc4572fb09/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
gs/java-compute/420/sponge_log.log
2020-11-15 01:28:13,852 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-15 01:28:14,062 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-420
nothing to commit, working tree clean
2020-11-15 01:28:14,208 synthtool [DEBUG] > Ensuring dependencies.
2020-11-15 01:28:14,219 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-15 01:28:14,219 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-15 01:28:14,223 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-15 01:28:24,083 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
Loading: 0 packages loaded
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Loading: 0 packages loaded
    currently loading: gapic/google/compute/v1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target @bazel_tools//tools/build_defs/pkg:build_tar: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.806s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-15 01:28:24,145 autosynth [ERROR] > Synthesis failed
2020-11-15 01:28:24,145 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-15 01:28:24,157 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-15 01:28:24,168 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/2d48c4d5-6fbc-4c75-843b-e95a63846307/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
0-11-15 14:27:49,838 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/421/sponge_log.log
2020-11-15 14:27:49,841 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-15 14:27:50,064 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-421
nothing to commit, working tree clean
2020-11-15 14:27:50,218 synthtool [DEBUG] > Ensuring dependencies.
2020-11-15 14:27:50,229 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-15 14:27:50,230 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-15 14:27:50,234 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-15 14:28:00,239 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 9.948s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-15 14:28:00,298 autosynth [ERROR] > Synthesis failed
2020-11-15 14:28:00,298 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-15 14:28:00,310 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-15 14:28:00,320 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/315dfd13-d43e-48d8-80e7-ed6d144096ea/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
-11-16 01:28:21,590 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/422/sponge_log.log
2020-11-16 01:28:21,593 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-16 01:28:21,817 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-422
nothing to commit, working tree clean
2020-11-16 01:28:21,973 synthtool [DEBUG] > Ensuring dependencies.
2020-11-16 01:28:21,983 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-16 01:28:21,984 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-16 01:28:21,988 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-16 01:28:32,231 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.186s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-16 01:28:32,298 autosynth [ERROR] > Synthesis failed
2020-11-16 01:28:32,298 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-16 01:28:32,316 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-16 01:28:32,327 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/39f5ce69-48ed-4006-9410-be96d98b7a8f/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
th [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/431/sponge_log.log
2020-11-16 14:28:37,612 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-16 14:28:37,841 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-431
nothing to commit, working tree clean
2020-11-16 14:28:38,005 synthtool [DEBUG] > Ensuring dependencies.
2020-11-16 14:28:38,020 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-16 14:28:38,021 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-16 14:28:38,026 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-16 14:28:48,772 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.672s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-16 14:28:48,857 autosynth [ERROR] > Synthesis failed
2020-11-16 14:28:48,857 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-16 14:28:48,875 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-16 14:28:48,890 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/2edce310-3b88-4a27-acbc-73bf8918c5a3/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
-11-17 01:28:10,831 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-compute/436/sponge_log.log
2020-11-17 01:28:10,833 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-17 01:28:11,047 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-436
nothing to commit, working tree clean
2020-11-17 01:28:11,197 synthtool [DEBUG] > Ensuring dependencies.
2020-11-17 01:28:11,207 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-17 01:28:11,208 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-17 01:28:11,212 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-17 01:28:21,498 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.229s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-17 01:28:21,554 autosynth [ERROR] > Synthesis failed
2020-11-17 01:28:21,554 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b2413cd chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#244)
2020-11-17 01:28:21,565 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-17 01:28:21,574 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/3f60f2f5-7b1d-4d8f-89f2-15b8ba5c4aea/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
log
2020-11-17 14:53:06,256 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-17 14:53:06,474 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-445
nothing to commit, working tree clean
2020-11-17 14:53:06,626 synthtool [DEBUG] > Ensuring dependencies.
2020-11-17 14:53:06,637 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-17 14:53:06,638 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-17 14:53:06,642 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-17 14:53:18,326 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded)
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.618s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-17 14:53:18,422 autosynth [ERROR] > Synthesis failed
2020-11-17 14:53:18,423 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 98b5455 chore(deps): bundle related dependencies to avoid conflicts during updates (#247)
2020-11-17 14:53:18,434 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-17 14:53:18,446 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/24248bb8-3dbe-4d0d-bb42-1ade5e7d2ff7/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/450/sponge_log.log
2020-11-18 01:36:37,862 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-18 01:36:38,070 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-450
nothing to commit, working tree clean
2020-11-18 01:36:38,217 synthtool [DEBUG] > Ensuring dependencies.
2020-11-18 01:36:38,227 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-18 01:36:38,228 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-18 01:36:38,231 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-18 01:36:48,618 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.333s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-18 01:36:48,678 autosynth [ERROR] > Synthesis failed
2020-11-18 01:36:48,678 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 98b5455 chore(deps): bundle related dependencies to avoid conflicts during updates (#247)
2020-11-18 01:36:48,689 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-18 01:36:48,698 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d95eb0e2-2e0c-4f6e-af68-a916bea589ef/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/464/sponge_log.log
2020-11-18 14:37:05,633 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-18 14:37:05,854 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-464
nothing to commit, working tree clean
2020-11-18 14:37:06,011 synthtool [DEBUG] > Ensuring dependencies.
2020-11-18 14:37:06,021 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-18 14:37:06,022 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-18 14:37:06,026 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-18 14:37:17,629 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
Loading: 0 packages loaded
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 11.548s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-18 14:37:17,696 autosynth [ERROR] > Synthesis failed
2020-11-18 14:37:17,696 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 98b5455 chore(deps): bundle related dependencies to avoid conflicts during updates (#247)
2020-11-18 14:37:17,708 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-18 14:37:17,718 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8c826c8b-8962-49dc-bd22-7540ab7ef1c8/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

Autosynth is still having trouble generating java-compute. :sob:

Here's the output from running `synth.py`:

```
BUG] > log_file_path: /tmpfs/src/logs/java-compute/466/sponge_log.log
2020-11-19 01:27:07,683 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-19 01:27:07,908 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-compute/synth.py.
On branch autosynth-466
nothing to commit, working tree clean
2020-11-19 01:27:08,062 synthtool [DEBUG] > Ensuring dependencies.
2020-11-19 01:27:08,073 synthtool [DEBUG] > Cloning discovery-artifact-manager.
2020-11-19 01:27:08,074 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/discovery-artifact-manager
2020-11-19 01:27:08,078 synthtool [DEBUG] > Generating code for: //gapic/google/compute/v1:google-cloud-compute-v1-java.
2020-11-19 01:27:18,561 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //gapic/google/compute/v1:google-cloud-compute-v1-java:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/fe1790ca0df67173702f70d5646b82f48f412b99.zip) = 419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "419f654d36998a51de2e0b79a67885ec48f8d803615ce66ec82cd15f54936730"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:17:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.54.0.zip) = 61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "61f813b2ad571a0ca59f9e28252a9958bb283198ab4dc827eb776c48dd13fc0d"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:54:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/128224d3c06a23fe55a32abd5d3760515974291d.zip) = cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cb2cd312a6c6560d641531879d51cc6e1b5a17f1eff485234d502eb37c681532"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:45:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip) = eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141
DEBUG: Rule 'com_google_googleapis' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "eccaa7c65f1ba78448c5bb3fbf2f1e8d60ec0f9b3c0e0eddf843cb7c7ea51141"
DEBUG: Call stack for the definition of repository 'com_google_googleapis' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:109:5
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/com_google_api_codegen/repositories.bzl:68:5
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:88:1
INFO: SHA256 (https://github.com/googleapis/protoc-java-resource-names-plugin/archive/7632facb0a1d9f76d9152eefe21422dc55e36524.zip) = 5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c
DEBUG: Rule 'com_google_protoc_java_resource_names_plugin' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5a1e32596021450457a0f9926e8bc601fb4f8d07e634afc5845719d3510a4c9c"
DEBUG: Call stack for the definition of repository 'com_google_protoc_java_resource_names_plugin' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a6a947520debde4a04620f960b938b83/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/discovery-artifact-manager/WORKSPACE:82:1
Analyzing: target //gapic/google/compute/v1:google-cloud-compute-v1-java (1 packages loaded, 0 targets configured)
ERROR: While resolving toolchains for target //gapic/google/compute/v1:compute_java_gapic_test: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
ERROR: Analysis of target '//gapic/google/compute/v1:google-cloud-compute-v1-java' failed; build aborted: invalid registered toolchain '@gapic_generator_python//:pyenv3_toolchain': no such package '@gapic_generator_python//': The repository '@gapic_generator_python' could not be resolved
INFO: Elapsed time: 10.423s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 21 targets configured)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-compute/synth.py", line 33, in <module>
    discogapic = True,
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 61, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 182, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//gapic/google/compute/v1:google-cloud-compute-v1-java']' returned non-zero exit status 1.
2020-11-19 01:27:18,636 autosynth [ERROR] > Synthesis failed
2020-11-19 01:27:18,636 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 98b5455 chore(deps): bundle related dependencies to avoid conflicts during updates (#247)
2020-11-19 01:27:18,648 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-19 01:27:18,660 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/da63bda4-2395-41ba-b245-551cea89aab1/targets/github%2Fsynthtool;config=default/tests;query=java-compute;failed=false).

--- a/synth.py
+++ b/synth.py
@@ -20,24 +20,6 @@ import synthtool.languages.java as java
 
 AUTOSYNTH_MULTIPLE_COMMITS = True
 
-gapic = gcp.GAPICBazel()
-
-service = 'compute'
-versions = ['v1']
-
-for version in versions:
-  library = gapic.java_library(
-      service=service,
-      version=version,
-      bazel_target=f'//gapic/google/{service}/{version}:google-cloud-{service}-{version}-java',
-      discogapic = True,
-  )
-
-  s.copy(library / f'gapic-google-cloud-{service}-{version}-java/src', 'src', required=True)
-  s.copy(library / f'google-cloud-{service}-{version}-java/gapic-google-cloud-{service}-{version}-java/src', 'src', required=True)
-
-  java.format_code('./src')
-
 java.common_templates(excludes=[
     'renovate.json'
 ])

